
<!DOCTYPE html>
<!--[if IEMobile 7 ]><html class="no-js iem7"><![endif]-->
<!--[if lt IE 9]><html class="no-js lte-ie8"><![endif]-->
<!--[if (gt IE 8)|(gt IEMobile 7)|!(IEMobile)|!(IE)]><!--><html class="no-js" lang="en"><!--<![endif]-->
<head>
  <meta charset="utf-8">
  <title>虎克的博客</title>
  <meta name="author" content="Hooker">

  
  <meta name="description" content="Biogeography: assessment and prospect There is general agreement among scientists that biodiversity is under assault on a global basis and that &hellip;">
  

  <!-- http://t.co/dKP3o1e -->
  <meta name="HandheldFriendly" content="True">
  <meta name="MobileOptimized" content="320">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  
  <link rel="canonical" href="http://Apiaceae.github.io/blog/page/8">
  <link href="/favicon.png" rel="icon">
  <link href="/stylesheets/screen.css" media="screen, projection" rel="stylesheet" type="text/css">
  <link href="/atom.xml" rel="alternate" title="虎克的博客" type="application/atom+xml">
  <script src="/javascripts/modernizr-2.0.js"></script>
  <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
  <script>!window.jQuery && document.write(unescape('%3Cscript src="./javascripts/lib/jquery.min.js"%3E%3C/script%3E'))</script>
  <script src="/javascripts/octopress.js" type="text/javascript"></script>
  <!--Fonts from Google"s Web font directory at http://google.com/webfonts -->
<link href="http://fonts.googleapis.com/css?family=PT+Serif:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css">
<link href="http://fonts.googleapis.com/css?family=PT+Sans:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css">

  

</head>

<body   >
  <header role="banner"><hgroup>
  <h1><a href="/">虎克的博客</a></h1>
  
    <h2>Enthusiasm Biogeography-Biodiversity Informatics-Data Sciences</h2>
  
</hgroup>

</header>
  <nav role="navigation"><ul class="subscription" data-subscription="rss">
  <li><a href="/atom.xml" rel="subscribe-rss" title="subscribe via RSS">RSS</a></li>
  
</ul>
  
<form action="http://google.com/search" method="get">
  <fieldset role="search">
    <input type="hidden" name="q" value="site:Apiaceae.github.io" />
    <input class="search" type="text" name="q" results="0" placeholder="Search"/>
  </fieldset>
</form>
  
<ul class="main-navigation">
  <li><a href="/">首页</a></li>
  <li><a href="/blog/archives">博客存档</a></li>
  <li><a href="/tag-cloud">标签</a></li>
    <li><a href="/about">关于</a></li>
</ul>

</nav>
  <div id="main">
    <div id="content">
      <div class="blog-index">
  
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2009/03/28/progress-conservation-biogeography/">保护生物地理学研究进展</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2009-03-28T00:00:00+08:00" pubdate data-updated="true">Mar 28<span>th</span>, 2009</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><img src="/images/2009/3/condore.jpg" alt="保护生物地理学" /></p>

<p><strong>Biogeography: assessment and prospect</strong></p>

<p>There is general agreement among scientists that biodiversity is under assault on a global basis and that species are being lost at a greatly enhanced rate. This article examines the role played by biogeographical science in the emergence of conservation guidance and makes the case for the recognition ofConservation Biogeography as a key subfield of conservation biology delimited as: the application of biogeographical principles, theories, and analyses, being those concerned with the distributional dynamics of taxa individually and collectively, to problems concerning the conservation of biodiversity.</p>

<p>Conservation biogeography thus encompasses both a substantial body of theory and analysis, and some of the most prominent planning works used in conservation. Considerable advances in conservation guidelines have been made over the last few decades by applying biogeographical methods and principles. Herein we provide a critical review focussed on the sensitivity to assumptions inherent in the applications we examine.</p>

<p>In particular, we focus on four inter-related factors:</p>

<p>(i) scale dependency (both spatial and temporal);
(ii) inadequacies in taxonomic and distributional data (the so-called Linnean and Wallacean shortfalls);
(iii) effects of model structure and parameterisation; and
(iv) inadequacies of theory.
These generic problems are illustrated by reference to studies ranging from the application of historical biogeography, through island biogeography, and complementarity analyses to bioclimatic envelope modelling. There is a great deal of uncertainty inherent in predictive analyses in conservation biogeography and this area in particular presents considerable challenges.</p>

<p>Protected area planning works and their resulting map outputs are amongst the most powerful and influential applications within conservation biogeography, and at the global scale are characterised by the production, by a small number of prominent NGOs, of bespoke schemes, which serve both to mobilise funds and channel efforts in a highly targeted fashion. We provide a simple typology of protected area planning works, with particular reference to the global scale, and provide a brief critique of some of their strengths and weaknesses. Finally, we discuss the importance, especially at regional scales, of developing more responsive analyses and models that integrate pattern (the compositionalist approach) and processes (the alist approach) such as range collapse and climate change, again noting the sensitivity of outcomes to starting assumptions. We make the case for the greater engagement of the biogeographical community in a programme of evaluation and refinement of all such schemes to test their robustness and their sensitivity to alternative conservation priorities and goals.</p>

<p>Keywords: Conservation biogeography, models, protected area works, scale, sensitivity analysis, uncertainty.</p>

<p class='post-footer'>
       original link:
       <a href='http://Apiaceae.github.io/blog/2009/03/28/progress-conservation-biogeography/'>http://Apiaceae.github.io/blog/2009/03/28/progress-conservation-biogeography/</a><br/>
       &nbsp;written by <a href='http://Apiaceae.github.io'>Hooker</a>
       &nbsp;posted at <a href='http://Apiaceae.github.io'>http://Apiaceae.github.io</a>
       </p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2009/03/27/phylogeography/">企鹅的谱系生物地理学</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2009-03-27T00:00:00+08:00" pubdate data-updated="true">Mar 27<span>th</span>, 2009</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><img src="/images/2009/3/35.jpg" alt="谱系地理1" /></p>

<p><img src="/images/2009/3/33.jpg" alt="谱系地理1" /></p>

<p>Multiple gene evidence for expansion of extant penguins out of Antarctica due to global cooling.</p>

<p>Classic problems in historical biogeography are <strong>where did penguins originate, and why are such mobile birds restricted to the Southern Hemisphere?</strong></p>

<p>Competing hypotheses posit they arose in tropical–warm temperate waters, species-diverse cool temperate regions, or in Gondwanaland w100 mya when it was further north.</p>

<p>To test these hypotheses we constructed a strongly supported phylogeny of extant penguins from 5851 bp of mitochondrial and nuclear DNA. Using Bayesian inference of ancestral areas we show that an Antarctic origin of extant taxa is highly likely, and that more derived taxa occur in lower latitudes. Molecular dating estimated penguins originated about 71 million years ago in Gondwanaland when it was further south and cooler.</p>

<p>Moreover, extant taxa are inferred to have originated in the Eocene, coincident with the extinction of the larger-bodied fossil taxa as global climate cooled. We hypothesize that, as Antarctica became ice-encrusted, modern penguins expanded via the circumpolar current to oceanic islands within the Antarctic Convergence, and later to the southern continents.</p>

<p>Thus, global cooling has had a major impact on penguin evolution, as it has on vertebrates generally. Penguins only reached cooler tropical waters in the Galapagos about 4 mya, and have not crossed the equatorial thermal barrier.</p>

<p><strong>Keywords: penguins; divergence time; biogeography; nuclear DNA; mitochondrial DNA</strong></p>

<p class='post-footer'>
       original link:
       <a href='http://Apiaceae.github.io/blog/2009/03/27/phylogeography/'>http://Apiaceae.github.io/blog/2009/03/27/phylogeography/</a><br/>
       &nbsp;written by <a href='http://Apiaceae.github.io'>Hooker</a>
       &nbsp;posted at <a href='http://Apiaceae.github.io'>http://Apiaceae.github.io</a>
       </p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2009/03/26/biogeograph-ldd/">极地植物的长距离散布</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2009-03-26T12:56:00+08:00" pubdate data-updated="true">Mar 26<span>th</span>, 2009</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><img src="/images/2009/3/barringl.gif" alt="LDD" /></p>

<p>由于缺乏对植物物种<strong>长距离散布</strong>频度好的量化估计，我们对它们的分布在气候变化条件下的响应情况的理解受到很大限制。通过分析分布与北极岛屿和Svalbard及其邻近地区的9个有花植物种的超过4000个取样的遗传变异分析，Alsos等的研究表明，自从最后大大冰期以来，在Svalbard岛屿居群的建立过程中，长距离散布的种类来源于多个地区。<strong>因此，长距离散布现象比我们所通常认为的那样要普遍的多，这反过来也表明这些种类分布范围的向北转移可能在全球变暖后发生地相当快速。</strong></p>

<p class='post-footer'>
       original link:
       <a href='http://Apiaceae.github.io/blog/2009/03/26/biogeograph-ldd/'>http://Apiaceae.github.io/blog/2009/03/26/biogeograph-ldd/</a><br/>
       &nbsp;written by <a href='http://Apiaceae.github.io'>Hooker</a>
       &nbsp;posted at <a href='http://Apiaceae.github.io'>http://Apiaceae.github.io</a>
       </p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2009/03/25/-nature-network/">有关自然网络体系 Nature Network</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2009-03-25T00:00:00+08:00" pubdate data-updated="true">Mar 25<span>th</span>, 2009</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><img src="/images/2009/3/vtheme90.jpg" alt="Nature Network" /></p>

<p><strong>Nature network</strong>是一个致力与让世界科学家通过互联网在全球和地区水平上就共同关心和感兴趣问题进行讨论的网站。</p>

<p>科学家通过相互的交流和讨论能够从中获得有价值的信息。网站有一些比较有趣的讨论和话题，比如：对如何成为一个好的研究生导师。</p>

<p class='post-footer'>
       original link:
       <a href='http://Apiaceae.github.io/blog/2009/03/25/-nature-network/'>http://Apiaceae.github.io/blog/2009/03/25/-nature-network/</a><br/>
       &nbsp;written by <a href='http://Apiaceae.github.io'>Hooker</a>
       &nbsp;posted at <a href='http://Apiaceae.github.io'>http://Apiaceae.github.io</a>
       </p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2009/03/24/power-tools-navicatce-for-mysql/">Navicat超强的数据格式转换和处理能力</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2009-03-24T20:15:00+08:00" pubdate data-updated="true">Mar 24<span>th</span>, 2009</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><img src="http://blog.cnpc.ac.cn/Blogs/image.axd?picture=2009%2f3%2f2009-03-24_174232.png" alt="" /></p>

<p class='post-footer'>
       original link:
       <a href='http://Apiaceae.github.io/blog/2009/03/24/power-tools-navicatce-for-mysql/'>http://Apiaceae.github.io/blog/2009/03/24/power-tools-navicatce-for-mysql/</a><br/>
       &nbsp;written by <a href='http://Apiaceae.github.io'>Hooker</a>
       &nbsp;posted at <a href='http://Apiaceae.github.io'>http://Apiaceae.github.io</a>
       </p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2009/03/24/-cybertaxonomy/">什么是Cybertaxonomy/etaxonomy</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2009-03-24T00:00:00+08:00" pubdate data-updated="true">Mar 24<span>th</span>, 2009</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><img src="/images/2009/3/vtheme111.jpg" alt="What is Cybertaxonomy" /></p>

<p><strong>Cybertaxonomy(taxonomy)</strong>: The fusion of taxonomy with cyberinfrastructure will transform descriptive taxonomy, creating a modern, efficient science to confront the biodiversity crisis. <em>Cybertaxonomy</em> will enable international teams of taxon experts, museums and herbaria to create and test taxonomic knowledge to assure humanity’s access to reliable information about earth’s species. The IISE and its partners are forging <em>cybertaxonomy</em> as the next logical link in a chain of scholarship unbroken since the <em>time of Linnaeus</em>.</p>

<p>The challenge is immense: discovering and describing millions of unknown species while at the same time continually critically testing 1.8 million already named species. International “knowledge communities,” each focused on a major taxon, will assemble and continually improve and expand all that is known of our world’s species.</p>

<p>We are conceiving a taxonomy-specific cyberinfrastructure that will become a worldwide “species observatory” capable of opening access to museum specimens, archived literature, collection data, digital instrumentation and colleagues to vastly expedite descriptive taxonomy. Resultant taxon “knowledge banks” will be the 21st century equivalent of revisions and monographs… the “high throughput” species testing mechanisms of the past. Digital tools, instruments and robots can ease constraints on nearly every phase of taxonomy from collecting and preparing specimens to sharing what is known through user-defined portals such as the Encyclopedia of Life (www.eol.org).</p>

<p>Knowing earth’s species and their myriad characters are fundamental to both evolutionary and environmental biology. The costs of building this new taxonomy are insignificant by comparison to the costs of ignorance as we face the biodiversity crisis.</p>

<p>The IISE will focus on four inter-related areas:</p>

<p><strong>(1) Species Inventories and Collection Development; </strong></p>

<p><strong>(2) Revisionary Taxonomy and Monography; </strong></p>

<p><strong>(3) Cyberinfrastructure; and</strong></p>

<p><strong>(4) the History and Philosophy of Taxonomy</strong></p>

<p class='post-footer'>
       original link:
       <a href='http://Apiaceae.github.io/blog/2009/03/24/-cybertaxonomy/'>http://Apiaceae.github.io/blog/2009/03/24/-cybertaxonomy/</a><br/>
       &nbsp;written by <a href='http://Apiaceae.github.io'>Hooker</a>
       &nbsp;posted at <a href='http://Apiaceae.github.io'>http://Apiaceae.github.io</a>
       </p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2009/03/23/geomapping-of-life-/">Geomapping of Life 我们需要它？</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2009-03-23T00:00:00+08:00" pubdate data-updated="true">Mar 23<span>rd</span>, 2009</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><img src="/images/2009/3/vtheme12.jpg" alt="COL首页" /></p>

<ul>
<li>Catalogue of Life</li>
</ul>


<p><img src="/images/2009/3/vtheme10.jpg" alt="TOL首页" /></p>

<ul>
<li>Tree of Life</li>
</ul>


<p><img src="/images/2009/3/vtheme11.jpg" alt="CBOL首页" /></p>

<ul>
<li>Barcoding of Life</li>
</ul>


<p><img src="/images/2009/3/vtheme9.jpg" alt="EOL首页" /></p>

<ul>
<li>Encyclopedia of Life</li>
</ul>


<p><strong>Geomapping of Life</strong></p>

<p class='post-footer'>
       original link:
       <a href='http://Apiaceae.github.io/blog/2009/03/23/geomapping-of-life-/'>http://Apiaceae.github.io/blog/2009/03/23/geomapping-of-life-/</a><br/>
       &nbsp;written by <a href='http://Apiaceae.github.io'>Hooker</a>
       &nbsp;posted at <a href='http://Apiaceae.github.io'>http://Apiaceae.github.io</a>
       </p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2009/03/22/new-book-sdm/">生态位模型预测新书上架</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2009-03-22T00:00:00+08:00" pubdate data-updated="true">Mar 22<span>nd</span>, 2009</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><img src="/images/2009/3/111.jpg" alt="书籍的封面" /></p>

<p><strong>Book Description</strong>: Using theory, applications, and examples of inferences, <strong>Niche Modeling</strong>: Predictions from Statistical Distributions demonstrates how to conduct and evaluate niche modeling projects in any area of application.</p>

<p>It features a series of theoretical and practical exercises for developing and evaluating niche models using the R statistics language. The author discusses applications of predictive modeling methods with reference to valid inferences from assumptions. He elucidates varied and simplified examples with rigor and completeness. Topics include geographic information systems, multivariate modeling, artificial intelligence methods, data handling, and information infrastructure.</p>

<p>Above all, successful niche modeling requires a deep understanding of the process of creating and using probability. Off-the-shelf statistical packages are tailored exactly to applications but can hide problematic complexities. Recipe book implementations fail to educate users in the details, assumptions, and pitfalls of analysis, but may be able to adapt to the specific needs of each study. Examining the sources of errors such as autocorrelation, bias, long term persistence, nonlinearity, circularity, and fraud, this seminal reference provides an understanding of the limitations and potential pitfalls of prediction, emphasizing the importance of avoiding errors.</p>

<p class='post-footer'>
       original link:
       <a href='http://Apiaceae.github.io/blog/2009/03/22/new-book-sdm/'>http://Apiaceae.github.io/blog/2009/03/22/new-book-sdm/</a><br/>
       &nbsp;written by <a href='http://Apiaceae.github.io'>Hooker</a>
       &nbsp;posted at <a href='http://Apiaceae.github.io'>http://Apiaceae.github.io</a>
       </p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2009/03/21/-lsid/">什么是生命科学识别码 LSID</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2009-03-21T00:00:00+08:00" pubdate data-updated="true">Mar 21<span>st</span>, 2009</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><img src="/images/2009/3/vtheme6.jpg" alt="LSID首页" /></p>

<p><strong>Life Science Identifiers (LSIDs)</strong> are persistent, location-independent, resource identifiers for uniquely naming biologically significant resources including species names, concepts, occurrences, genes or proteins, or data objects that encode information about them. To put it simply, LSIDs are a way to identify and locate pieces of biological information on the web.</p>

<p>The <em>LSID</em> concept introduces a straightforward approach to naming and identifying data resources stored in multiple, distributed data stores in a manner that overcomes the limitations of naming schemes in use today. Almost every public, internal, or department-level data store today has its own way of naming individual data resources, making integration between different data sources a tedious, never-ending chore for informatics developers and researchers.</p>

<p>By defining a simple, common way to identify and access biologically significant data, whether that data is stored in files, relational databases, in applications, or in internal or public data sources, LSID provides a naming standard underpinning for wide-area science and interoperability.</p>

<p>The Life Sciences Identifier is an <a href="http://www.omg.org/">Object Management Group (OMG)</a> Final Adopted Specification developed by the <a href="http://www.omg.org/lsr/">Life Sciences Research (LSR)</a>.</p>

<p class='post-footer'>
       original link:
       <a href='http://Apiaceae.github.io/blog/2009/03/21/-lsid/'>http://Apiaceae.github.io/blog/2009/03/21/-lsid/</a><br/>
       &nbsp;written by <a href='http://Apiaceae.github.io'>Hooker</a>
       &nbsp;posted at <a href='http://Apiaceae.github.io'>http://Apiaceae.github.io</a>
       </p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2009/03/20/american-nsfc-bi/">美国自然科学基金对数字化标本平台的支持</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2009-03-20T00:00:00+08:00" pubdate data-updated="true">Mar 20<span>th</span>, 2009</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><img src="/images/2009/3/vtheme7.jpg" alt="项目首页" /></p>

<p>The US National Science Foundation has awarded a <strong>Research Coordination Grant</strong> entitled: <strong>Building a National Community of Natural History Collections</strong>.</p>

<p>The purpose of the RCN is to build a strong scientific community to better serve researchers in systematics, biodiversity, and other fields that use natural history collections.</p>

<p>Three professional organizations, <em>the American Institute of Biological Sciences</em>, <em>the Natural Science Collections Alliance</em>, and <em>the Society for the Preservation of Natural History Collections</em> will provide strong support as partners in the RCN, and many other societies have already built ties with the RCN by appointing Core Participants.</p>

<p>The RCN has the following objectives:</p>

<ul>
<li><p>Identify the institutions and people that will define our community and facilitate dialogue among them about how to better serve the needs of researchers,</p></li>
<li><p>identify major opportunities and challenges in the current environment and foreseeable future and develop a strategic plan for the future of collections research,</p></li>
<li><p>determine how to strengthen and modernize the role of collections in education and outreach, and</p></li>
<li><p>identify the primary needs of collections regarding care, curation, storage, and accessibility.</p></li>
</ul>


<p>The objectives will be met with a series of workshops, symposia, internships, and a website, <a href="http://www.collectionsweb.org/">www.CollectionsWeb.org</a>, which will provide a forum for interaction, host workshop reports, provide information on ways to become involved in the RCN, and provide links to resources for NHC.</p>

<p>Several major community resources will be developed, including a catalogue of NHC, a survey of the status of NHC, a register of curatorial expertise, and an inventory of innovative and successful educational programs.</p>

<p>Broadening minority participation in collections activities is a goal of the RCN.</p>

<p>The initial steering committee members are Henry L. Bart (Tulane University, vertebrate systematics), Meredith Blackwell (Louisiana State University, fungal systematics), L. Alan Prather (Michigan State University, plant
systematics) and James B. Woolley (Texas A &amp; M University, invertebrate systematics).</p>

<p>Announcements of upcoming events, opportunities to get involved, and reports will be posted to the website, so please check back often. Your input is essential!  Contact us at <a href="http://www.collectionsweb.org/about/steering_committee.html.">http://www.collectionsweb.org/about/steering_committee.html.</a></p>

<p class='post-footer'>
       original link:
       <a href='http://Apiaceae.github.io/blog/2009/03/20/american-nsfc-bi/'>http://Apiaceae.github.io/blog/2009/03/20/american-nsfc-bi/</a><br/>
       &nbsp;written by <a href='http://Apiaceae.github.io'>Hooker</a>
       &nbsp;posted at <a href='http://Apiaceae.github.io'>http://Apiaceae.github.io</a>
       </p>

</div>
  
  


    </article>
  
  <div class="pagination">
    
      <a class="prev" href="/blog/page/9/">&larr; Older</a>
    
    <a href="/blog/archives">Blog Archives</a>
    
    <a class="next" href="/blog/page/7/">Newer &rarr;</a>
    
  </div>
</div>
<aside class="sidebar">
  
    <section>
  <h1>About Me</h1>
  <img src="/images/profile.jpg" alt="">
  <p>混在城市心在农村</p>
</section>
<section>
  <h2>What did I read</h2>
  <div>
    <script type="text/javascript" src="http://www.douban.com/service/badge/57046744/?show=collection&amp;select=random&amp;n=8&amp;columns=2&amp;cat=book" ></script>
    </div>
</section>
<section>
  <h1>Recent Posts</h1>
  <ul id="recent_posts">
    
      <li class="post">
        <a href="/blog/2013/09/26/fifty-best-sites-build-on-ruby-on-rails/">最好的50个用Ruby on Rails开发的网站</a>
      </li>
    
      <li class="post">
        <a href="/blog/2013/09/26/why-we-live-in-big-city/">我？你？他？身处大城市的十大尴尬</a>
      </li>
    
      <li class="post">
        <a href="/blog/2013/09/23/pry-relace-rails-console/">使用PRY代替RAILS CONSOLE</a>
      </li>
    
      <li class="post">
        <a href="/blog/2013/09/03/nginx/">Nginx内核参数相关的优化设定说明</a>
      </li>
    
      <li class="post">
        <a href="/blog/2013/09/03/lamp/">LAMP系统性能调优</a>
      </li>
    
  </ul>
</section>
<section>
  <h1>Categories</h1>
  <ul id="categories">
    <li class='category'><a href='/blog/categories/asp-net/'>ASP.NET (4)</a></li>
<li class='category'><a href='/blog/categories/biodiversity/'>Biodiversity (1)</a></li>
<li class='category'><a href='/blog/categories/biodiversity-informatics/'>Biodiversity Informatics (5)</a></li>
<li class='category'><a href='/blog/categories/git/'>Git (1)</a></li>
<li class='category'><a href='/blog/categories/informatics/'>Informatics (1)</a></li>
<li class='category'><a href='/blog/categories/linux/'>Linux (8)</a></li>
<li class='category'><a href='/blog/categories/ruby/'>Ruby (1)</a></li>
<li class='category'><a href='/blog/categories/data/'>data (1)</a></li>
<li class='category'><a href='/blog/categories/linux/'>linux (3)</a></li>
<li class='category'><a href='/blog/categories/ruby/'>ruby (1)</a></li>
<li class='category'><a href='/blog/categories/sciences/'>sciences (1)</a></li>
<li class='category'><a href='/blog/categories/伞形科/'>伞形科 (1)</a></li>
<li class='category'><a href='/blog/categories/分类学/'>分类学 (4)</a></li>
<li class='category'><a href='/blog/categories/工具/'>工具 (1)</a></li>
<li class='category'><a href='/blog/categories/技巧/'>技巧 (1)</a></li>
<li class='category'><a href='/blog/categories/数据库/'>数据库 (1)</a></li>
<li class='category'><a href='/blog/categories/物种分布模型/'>物种分布模型 (1)</a></li>
<li class='category'><a href='/blog/categories/生物地理/'>生物地理 (2)</a></li>
<li class='category'><a href='/blog/categories/生物地理学/'>生物地理学 (3)</a></li>
<li class='category'><a href='/blog/categories/生物多样性信息学/'>生物多样性信息学 (7)</a></li>
<li class='category'><a href='/blog/categories/科学社会网络/'>科学社会网络 (1)</a></li>
<li class='category'><a href='/blog/categories/随笔/'>随笔 (4)</a></li>

  </ul>
</section>
<section>
  <h1>Tags</h1>
  <ul class="tag-cloud">
    <a style="font-size: 125%" href="/tags/asp-dot-net/">ASP.NET</a>
<a style="font-size: 180%" href="/tags/biodiversity-informatics/">Biodiversity Informatics</a>
<a style="font-size: 70%" href="/tags/lsid/">LSID</a>
<a style="font-size: 125%" href="/tags/linux/">Linux</a>
<a style="font-size: 70%" href="/tags/mapping/">Mapping</a>
<a style="font-size: 70%" href="/tags/sdm/">SDM</a>
<a style="font-size: 70%" href="/tags/api/">api</a>
<a style="font-size: 70%" href="/tags/etaxonomy/">etaxonomy</a>
<a style="font-size: 125%" href="/tags/sed/">sed</a>
<a style="font-size: 125%" href="/tags/taxonomy/">taxonomy</a>
<a style="font-size: 180%" href="/tags/ubuntu/">ubuntu</a>
<a style="font-size: 157%" href="/tags/xin-qing/">心情</a>
<a style="font-size: 157%" href="/tags/sheng-huo/">生活</a>
<a style="font-size: 180%" href="/tags/sheng-wu-duo-yang-xing-xin-xi-xue/">生物多样性信息学</a>
<a style="font-size: 157%" href="/tags/sui-bi/">随笔</a>

  </ul>
</section><section>
  
  <h1>Visitors</h1>
    <div id="clustrmaps-widget"></div><script type="text/javascript">
var _clustrmaps = {'url' : 'http://apiaceae.github.io', 'user' : 1114201, 'server' : '2', 'id' : 'clustrmaps-widget', 'version' : 1, 'date' : '2013-09-25', 'lang' : 'en', 'corners' : 'square' };
    (function ()
    { var s = document.createElement('script'); s.type = 'text/javascript'; s.async = true; s.src = 'http://www2.clustrmaps.com/counter/map.js'; var x = document.getElementsByTagName('script')[0]; x.parentNode.insertBefore(s, x);})();
    </script><noscript><a href="http://www2.clustrmaps.com/user/195110059"><img src="http://www2.clustrmaps.com/stats/maps-no_clusters/apiaceae.github.io-thumb.jpg" alt="Locations of visitors to this page"></a></noscript>

    <a href="http://info.flagcounter.com/wAs5"><img src="http://s07.flagcounter.com/count/wAs5/bg_FFFFFF/txt_000000/border_CCCCCC/columns_2/maxflags_12/viewers_0/labels_0/pageviews_0/flags_0/" alt="Flag Counter" border="0"></a>


</section>




  
</aside>

    </div>
  </div>
  <footer role="contentinfo"><p>
  Copyright &copy; 2013 - Hooker -
  <span class="credit">Powered by <a href="http://octopress.org">Octopress</a></span>
</p>

</footer>
  





  <script type="text/javascript">
    (function() {
      var script = document.createElement('script'); script.type = 'text/javascript'; script.async = true;
      script.src = 'https://apis.google.com/js/plusone.js';
      var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(script, s);
    })();
  </script>



  <script type="text/javascript">
    (function(){
      var twitterWidgets = document.createElement('script');
      twitterWidgets.type = 'text/javascript';
      twitterWidgets.async = true;
      twitterWidgets.src = '//platform.twitter.com/widgets.js';
      document.getElementsByTagName('head')[0].appendChild(twitterWidgets);
    })();
  </script>





</body>
</html>
