
<!DOCTYPE html>
<!--[if IEMobile 7 ]><html class="no-js iem7"><![endif]-->
<!--[if lt IE 9]><html class="no-js lte-ie8"><![endif]-->
<!--[if (gt IE 8)|(gt IEMobile 7)|!(IEMobile)|!(IE)]><!--><html class="no-js" lang="en"><!--<![endif]-->
<head>
  <meta charset="utf-8">
  <title>虎克的博客</title>
  <meta name="author" content="Hooker">

  
  <meta name="description" content="&nbsp;EOL程序的视图发生了很多的变化，已经不再是使用简单的 .rhtml 文件格式了。使用了一个一种以 .haml 结尾的文件。见下图，很奇怪的语法和格式。&nbsp;Haml 是一种用来描述XHTML 的标记语言。 Haml 容易阅读，让开发人员避免把代码写在 XHTML 模板中。 &hellip;">
  

  <!-- http://t.co/dKP3o1e -->
  <meta name="HandheldFriendly" content="True">
  <meta name="MobileOptimized" content="320">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  
  <link rel="canonical" href="http://Apiaceae.github.io/blog/page/8">
  <link href="/favicon.png" rel="icon">
  <link href="/stylesheets/screen.css" media="screen, projection" rel="stylesheet" type="text/css">
  <link href="/atom.xml" rel="alternate" title="虎克的博客" type="application/atom+xml">
  <script src="/javascripts/modernizr-2.0.js"></script>
  <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
  <script>!window.jQuery && document.write(unescape('%3Cscript src="./javascripts/lib/jquery.min.js"%3E%3C/script%3E'))</script>
  <script src="/javascripts/octopress.js" type="text/javascript"></script>
  <!--Fonts from Google"s Web font directory at http://google.com/webfonts -->
<link href="http://fonts.googleapis.com/css?family=PT+Serif:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css">
<link href="http://fonts.googleapis.com/css?family=PT+Sans:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css">

  

</head>

<body   >
  <header role="banner"><hgroup>
  <h1><a href="/">虎克的博客</a></h1>
  
    <h2>Enthusiasm Biogeography-Biodiversity Informatics-Data Sciences</h2>
  
</hgroup>

</header>
  <nav role="navigation"><ul class="subscription" data-subscription="rss">
  <li><a href="/atom.xml" rel="subscribe-rss" title="subscribe via RSS">RSS</a></li>
  
</ul>
  
<form action="http://google.com/search" method="get">
  <fieldset role="search">
    <input type="hidden" name="q" value="site:Apiaceae.github.io" />
    <input class="search" type="text" name="q" results="0" placeholder="Search"/>
  </fieldset>
</form>
  
<ul class="main-navigation">
  <li><a href="/">Blog</a></li>
  <li><a href="/blog/archives">Archives</a></li>
  <li><a href="/">About me</a></li> 
</ul>

</nav>
  <div id="main">
    <div id="content">
      <div class="blog-index">
  
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2010/01/31/e4bb80e4b988e698afhamle69687e4bbb6e6a0bce5bc8f/">什么是Haml文件格式</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2010-01-31T00:00:00+08:00" pubdate data-updated="true">Jan 31<span>st</span>, 2010</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p style="clear: both"><a href="/Blogs/images/2010-01-31_2208.png" class="image-link"><img style="display: inline; float: left; margin-top: 0px; margin-right: 10px; margin-bottom: 10px; margin-left: 0px" class="linked-to-original" src="/Blogs/images/2010-01-31_2208-thumb.png" alt="" width="380" height="258" align="left" /></a></p><p style="clear: both">&nbsp;</p><p style="clear: both">EOL程序的视图发生了很多的变化，已经不再是使用简单的 .rhtml 文件格式了。使用了一个一种以 .haml 结尾的文件。见下图，很奇怪的语法和格式。</p><p style="clear: both">&nbsp;</p><p style="clear: both">Haml 是一种用来描述XHTML 的标记语言。 Haml 容易阅读，让开发人员避免把代码写在 XHTML 模板中。他同样遵守 Ruby On Rails 的一些核心原则，比如：不要重复自己；代码优雅 ；容易阅读 。<br />例如我们有如下的 XHTML文件： </p><p>&nbsp;</p><p>&nbsp;</p><p style="clear: both"><a href="/Blogs/images/2010-01-31_2230.png" class="image-link"><img style="display: inline; float: left; margin-top: 0px; margin-right: 10px; margin-bottom: 10px; margin-left: 0px" class="linked-to-original" src="/Blogs/images/2010-01-31_2230-thumb.png" alt="" width="380" height="382" align="left" /></a><br />还有其它一下有用的学习资源：<br /><a href="http://wiki.rubyonrails.org/howtos/templates/haml" target="_blank" title="Haml Wiki">http://wiki.rubyonrails.org/howtos/templates/haml</a><br /><a href="http://haml-lang.com/" target="_blank" title="Haml-lang">http://haml-lang.com/</a><br /></p>
</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2010/01/25/e7949fe789a9e5a49ae6a0b7e680a7e4bfa1e681afe5ada6e696b0e69687e7aba0/">生物多样性信息学新文章</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2010-01-25T00:00:00+08:00" pubdate data-updated="true">Jan 25<span>th</span>, 2010</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><a href="http://blog.cnpc.ac.cn/Blogs/images/WindowsLiveWriter/bf513f73283b/2805E44B/20100125_00012.gif"><img style="border-bottom: 0px; border-left: 0px; display: inline; border-top: 0px; border-right: 0px" title="2010-01-25_00012" border="0" alt="2010-01-25_00012" src="http://blog.cnpc.ac.cn/Blogs/images/WindowsLiveWriter/bf513f73283b/47B4BE13/20100125_00012_thumb.gif" width="129" height="44" /></a> </p>  <p>BMC Bioinformatics最近出版了一期增刊，专门讨论生物多样性信息学(Biodiversity Informatcs: 简称BI)</p>  <p>&#160;<a title="http://www.biomedcentral.com/1471-2105/10?issue=S14" href="http://www.biomedcentral.com/1471-2105/10?issue=S14">http://www.biomedcentral.com/1471-2105/10?issue=S14</a></p>  <p><a href="http://blog.cnpc.ac.cn/Blogs/images/WindowsLiveWriter/bf513f73283b/75A210CB/20100125_00013.gif"><img style="border-bottom: 0px; border-left: 0px; display: inline; border-top: 0px; border-right: 0px" title="2010-01-25_00013" border="0" alt="2010-01-25_00013" src="http://blog.cnpc.ac.cn/Blogs/images/WindowsLiveWriter/bf513f73283b/2799B156/20100125_00013_thumb.gif" width="638" height="695" /></a></p>
</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2010/01/20/linux-ubuntu804e5b8b8e794a8e6938de4bd9ce591bde4bba4/">Linux Ubuntu8.04常用操作命令</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2010-01-20T15:30:00+08:00" pubdate data-updated="true">Jan 20<span>th</span>, 2010</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><em>1. 如何完全的卸载MySQL</em></p>

<figure class='code'><div class="highlight"><table><tr><td class="gutter"><pre class="line-numbers"><span class='line-number'>1</span>
</pre></td><td class='code'><pre><code class=''><span class='line'>sudo apt-get --purge remove mysql-server-5.1 mysql-common mysql-client libmysqlclient15-dev libmysql-ruby</span></code></pre></td></tr></table></div></figure>


<p>卸载过程中会询问你是否要卸载所有相关的配置文件已经数据库，选择Yes。</p>

<p><em>2. 安装完MySQL后如何设置root用户的密码</em></p>

<figure class='code'><div class="highlight"><table><tr><td class="gutter"><pre class="line-numbers"><span class='line-number'>1</span>
</pre></td><td class='code'><pre><code class=''><span class='line'>mysqladmin -u root password new-password</span></code></pre></td></tr></table></div></figure>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2009/04/01/-/">最早的生物地理学地图</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2009-04-01T00:00:00+08:00" pubdate data-updated="true">Apr 1<span>st</span>, 2009</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><img src="/images/2009/3/vtheme999.jpg" alt="最早的生物地理学地图" /></p>

<p><strong>The first biogeographical map</strong></p>

<p>The first biogeographical map was published in the third edition of the Flore franc竌ise by Lamarck and Candolle in 1805, the same year in which Humboldt’s famous Essai sur la Geographie appeared. Lamarck and Candolle’s map marks the beginning of a deive or classificatory biogeography focusing on the study of biota rather than on the distributional pathways of taxa. The map is relevant because it heralds the beginning of the creation of biogeographical maps popularized by zoogeographers in the mid- to late nineteenth century together with the study of biogeographical regions.</p>

<p>see more details on <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2699.2006.01477.x/full">Journal of Biogeography</a></p>

<p>Included file &lsquo;JB/setup&rsquo; not found in _includes directory</p>
</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2009/03/30/-/">分类学是生物多样性研究的基础</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2009-03-30T00:00:00+08:00" pubdate data-updated="true">Mar 30<span>th</span>, 2009</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><img src="/assets/images/2009/3/vtheme4.jpg" alt="分类学和生物多样性" /></p>

<p><strong>Biodiversity Update–Progress in Taxonomy</strong></p>

<p>Taxonomy and systematic biology responded to the “biodiversity crisis” and the Convention on Biological Diversity (CBD) by defining three major challenges: completing the inventory of life, discovering evolutionary relationships through phylogenetic analysis, and providing information via the Internet (2). Progress in reconstructing the “Tree of Life” (3, 4) is effective and well coordinated, with phylogenetics increasingly integrated into ecology, biogeography, and developmental biology (5).</p>

<p>However, we are far from placing each species in its phylogenetic context, because the inventory itself lags behind. Out of a minimum of 4 million species, about 1.7 million (6) have so far been recognized through comparative studies of preserved specimens in reference collections. Information technology can accelerate this work (7), but the core of the discipline will remain. Specimens will be needed to confer repeatability on the hypothesis that a particular element of biodiversity constitutes a species and for taxonomists to identify new species and unfamiliar organisms.</p>

<p>As the World Conference on Sustainable Development (8) convenes, do we know enough either to conserve biodiversity or to make the best use of it? The Global Biodiversity Information Facility (GBIF) is a vital step toward accessible species-level information (9, 10). It establishes an informatic architecture for synthesizing knowledge, capable of uniting taxonomic initiatives such as Species 2000 (11) with regional databases (12, 13). If information that exists only in biological collections can also be computerized, users everywhere will share the benefits (14). Retrospective digitization of collections requires most investment, after which data capture and stewardship can be linked to ongoing curatorial work. Access to images of the type specimens that define species is one of the highest priorities, and the All Species Foundation’s proposal to digitize 50% of them in 5 years (15) poses no technical challenges once funding is secured. Surprisingly, digitizing collections is a contentious topic where the priorities of conservationists and taxonomists differ. The former tend to favor biodiversity conservation databases (16) based on field observations of the status of species. The latter recognize the importance of such efforts that build on the traditions of biological recording but point out that they are only practicable for groups where taxonomic work has progressed to provide detailed species level information.</p>

<p>Otherwise, what can be observed and recorded? This, together with the availability of skillful amateurs equipped with excellent field guides, is why birds are useful indicators of trends in biodiversity, but microalgae, although suitably widespread, abundant, and sensitive to change, are not. Historical data from collections already play a powerful role in planning and development in Mexico (17) and could do so universally. Biological collections may represent an inadequate sample of past and present biota, but they are the only identified and authenticated sample we have. Biological recording and collections are complementary, not competing, sources of knowledge about the world, with the latter enabling repeatable observations and entirely new kinds of investigation. Prospects are not good for the first and most fundamental challenge: a global inventory of life on Earth.</p>

<p>E.O.Wilson diagnosed the problem as the limited capacity of collections institutions and the shortage of taxonomists (18), conclusions shared by a recent UK review (19). Wilson estimated the cost at around $5 billion, about the same as the Human Genome Project. Success would require increased taxonomic capacity, in terms of people and collections, around the world. These are the goals of the Global Taxonomy Initiative (20), for which developing countries can receive resources from the Global Environment Facility (GEF) (21).</p>

<p>The All Species Foundation also aspires to complete the global inventory within “a single human generation” but focuses on developing sources of private funding. Redirection of existing resources might be more effective. Developed countries mainly contribute to the international objectives of the CBD through the GEF, which has so far committed $3.86 billion to biodiversity projects, especially in protected areas. They might achieve more by targeting a significant proportion of the resources that go to the GEF directly to their own taxonomic institutions, enabling them to reap the benefits of the long-term investment that created them. There can be little doubt that such resources could be deployed rapidly and effectively. The National Science Foundation has shown how quickly taxonomic expertise can be generated even in groups where it was scarce (22). Strategic plans, and the networks to implement them, have been established in many regions (23-25) with a few countries alrea***mitted to national inventories (26, 27). The documentation of life on Earth, on which our own well-being ultimately depends, surely deserves to be among our most urgent priorities for investment.</p>

<p><strong>Science 11 October 2002: Vol. 298. no. 5592, p. 365 DOI: 10.1126/science.1075026</strong></p>

<p>Included file &lsquo;JB/setup&rsquo; not found in _includes directory</p>
</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2009/03/29/">生物地理学空间单元的分类</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2009-03-29T00:00:00+08:00" pubdate data-updated="true">Mar 29<span>th</span>, 2009</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><img src="/assets/images/2009/3/9780521360401.jpg" alt="生物地理学单元" /></p>

<p><strong>Biochore classification and nomenclature in paleobiogeography: an attempt at order</strong></p>

<p>Gerd E.G. Westermann</p>

<p>Abstract: Paleobiogeographic terminology has increased dramatically in recent decades, but the absence of ‘rules’ or even a guide has resulted in confusion and misunderstanding. The Friends of Paleobiogeography are attempting to solve problems related to classification, definition and nomenclature of the biogeographic units (biochores), in conjunction with neobiogeographers. Historic and current developments are discussed. Biochores are highly dynamic units that not only expand and shrink in range, but also change in rank (tier) through time. They should be based on as many higher taxa as possible representing the biota, but from a single major biome, e.g. pelagic versus benthic; different biomes may need distinct sets of biochores. The following tentative ‘rules’ are proposed for a prospective guide for the distinction, ranking and naming of biochores, with emphasis on stability.</p>

<p>Definition: a biochore is defined by the overall endemism of its biota (not particular taxa) within a geographic envelope around a core area, whereas biochore boundaries are defined by the temporary range limits of their constituent endemic taxa.</p>

<p>Ranks (tiers): the biochores ranks are, with ‘bold’ for obligatory use and an additional tentative rank for exceptional conditions in brackets: [Superrealm], Realm, Subrealm, Province, Subprovince; ‘region’ is for informal use. Ranks scale with the degree of endemism as well as duration and biota distribution (range).</p>

<p>A typical region or chorotype and a typical stage (age) or chronotype are designated for each biochore.</p>

<p>Nomenclature: biochore names are geographic and related terms, not taxa-based. Nomenclatural priority begins with Uhlig [Mitt. Geol. Ges. Wien 4 (3) (1911) 229–448]. ‘Rules of Homology’ and ‘Synonymy’ apply in guide form only, e.g. long-term disuse of name or poor definition of biota invalidates synonym or homonym even if senior.</p>

<p>Keywords: biogeography; classification; nomenclature; paleobiogeography; terminology</p>

<p>Included file &lsquo;JB/setup&rsquo; not found in _includes directory</p>
</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2009/03/28/">保护生物地理学研究进展</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2009-03-28T00:00:00+08:00" pubdate data-updated="true">Mar 28<span>th</span>, 2009</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><img src="/assets/images/2009/3/condore.jpg" alt="保护生物地理学" /></p>

<p><strong>Biogeography: assessment and prospect</strong></p>

<p>There is general agreement among scientists that biodiversity is under assault on a global basis and that species are being lost at a greatly enhanced rate. This article examines the role played by biogeographical science in the emergence of conservation guidance and makes the case for the recognition ofConservation Biogeography as a key subfield of conservation biology delimited as: the application of biogeographical principles, theories, and analyses, being those concerned with the distributional dynamics of taxa individually and collectively, to problems concerning the conservation of biodiversity.</p>

<p>Conservation biogeography thus encompasses both a substantial body of theory and analysis, and some of the most prominent planning works used in conservation. Considerable advances in conservation guidelines have been made over the last few decades by applying biogeographical methods and principles. Herein we provide a critical review focussed on the sensitivity to assumptions inherent in the applications we examine.</p>

<p>In particular, we focus on four inter-related factors:</p>

<p>(i) scale dependency (both spatial and temporal);
(ii) inadequacies in taxonomic and distributional data (the so-called Linnean and Wallacean shortfalls);
(iii) effects of model structure and parameterisation; and
(iv) inadequacies of theory.
These generic problems are illustrated by reference to studies ranging from the application of historical biogeography, through island biogeography, and complementarity analyses to bioclimatic envelope modelling. There is a great deal of uncertainty inherent in predictive analyses in conservation biogeography and this area in particular presents considerable challenges.</p>

<p>Protected area planning works and their resulting map outputs are amongst the most powerful and influential applications within conservation biogeography, and at the global scale are characterised by the production, by a small number of prominent NGOs, of bespoke schemes, which serve both to mobilise funds and channel efforts in a highly targeted fashion. We provide a simple typology of protected area planning works, with particular reference to the global scale, and provide a brief critique of some of their strengths and weaknesses. Finally, we discuss the importance, especially at regional scales, of developing more responsive analyses and models that integrate pattern (the compositionalist approach) and processes (the alist approach) such as range collapse and climate change, again noting the sensitivity of outcomes to starting assumptions. We make the case for the greater engagement of the biogeographical community in a programme of evaluation and refinement of all such schemes to test their robustness and their sensitivity to alternative conservation priorities and goals.</p>

<p>Keywords: Conservation biogeography, models, protected area works, scale, sensitivity analysis, uncertainty.</p>

<p>Included file &lsquo;JB/setup&rsquo; not found in _includes directory</p>
</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2009/03/27/">企鹅的谱系生物地理学</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2009-03-27T00:00:00+08:00" pubdate data-updated="true">Mar 27<span>th</span>, 2009</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><img src="/assets/images/2009/3/35.jpg" alt="谱系地理1" /></p>

<p><img src="/assets/images/2009/3/33.jpg" alt="谱系地理1" /></p>

<p>Multiple gene evidence for expansion of extant penguins out of Antarctica due to global cooling.</p>

<p>Classic problems in historical biogeography are <strong>where did penguins originate, and why are such mobile birds restricted to the Southern Hemisphere?</strong></p>

<p>Competing hypotheses posit they arose in tropical–warm temperate waters, species-diverse cool temperate regions, or in Gondwanaland w100 mya when it was further north.</p>

<p>To test these hypotheses we constructed a strongly supported phylogeny of extant penguins from 5851 bp of mitochondrial and nuclear DNA. Using Bayesian inference of ancestral areas we show that an Antarctic origin of extant taxa is highly likely, and that more derived taxa occur in lower latitudes. Molecular dating estimated penguins originated about 71 million years ago in Gondwanaland when it was further south and cooler.</p>

<p>Moreover, extant taxa are inferred to have originated in the Eocene, coincident with the extinction of the larger-bodied fossil taxa as global climate cooled. We hypothesize that, as Antarctica became ice-encrusted, modern penguins expanded via the circumpolar current to oceanic islands within the Antarctic Convergence, and later to the southern continents.</p>

<p>Thus, global cooling has had a major impact on penguin evolution, as it has on vertebrates generally. Penguins only reached cooler tropical waters in the Galapagos about 4 mya, and have not crossed the equatorial thermal barrier.</p>

<p><strong>Keywords: penguins; divergence time; biogeography; nuclear DNA; mitochondrial DNA</strong></p>

<p>Included file &lsquo;JB/setup&rsquo; not found in _includes directory</p>
</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2009/03/26/">极地植物的长距离散布</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2009-03-26T00:00:00+08:00" pubdate data-updated="true">Mar 26<span>th</span>, 2009</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><img src="/assets/images/2009/3/barringl.gif" alt="LDD" /></p>

<p>由于缺乏对植物物种<strong>长距离散布</strong>频度好的量化估计，我们对它们的分布在气候变化条件下的响应情况的理解受到很大限制。通过分析分布与北极岛屿和Svalbard及其邻近地区的9个有花植物种的超过4000个取样的遗传变异分析，Alsos等的研究表明，自从最后大大冰期以来，在Svalbard岛屿居群的建立过程中，长距离散布的种类来源于多个地区。<strong>因此，长距离散布现象比我们所通常认为的那样要普遍的多，这反过来也表明这些种类分布范围的向北转移可能在全球变暖后发生地相当快速。</strong></p>

<p>Included file &lsquo;JB/setup&rsquo; not found in _includes directory</p>
</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2009/03/25/-nature-network/">有关自然网络体系 Nature Network</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2009-03-25T00:00:00+08:00" pubdate data-updated="true">Mar 25<span>th</span>, 2009</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><img src="/assets/images/2009/3/vtheme90.jpg" alt="Nature Network" /></p>

<p><strong>Nature network</strong>是一个致力与让世界科学家通过互联网在全球和地区水平上就共同关心和感兴趣问题进行讨论的网站。</p>

<p>科学家通过相互的交流和讨论能够从中获得有价值的信息。网站有一些比较有趣的讨论和话题，比如：对如何成为一个好的研究生导师。</p>

<p>Included file &lsquo;JB/setup&rsquo; not found in _includes directory</p>
</div>
  
  


    </article>
  
  <div class="pagination">
    
      <a class="prev" href="/blog/page/9/">&larr; Older</a>
    
    <a href="/blog/archives">Blog Archives</a>
    
    <a class="next" href="/blog/page/7/">Newer &rarr;</a>
    
  </div>
</div>
<aside class="sidebar">
  
    <section>
  <h1>About Me</h1>
  <p>A little something about me.</p>
</section>
<section>
  <h1>Recent Posts</h1>
  <ul id="recent_posts">
    
      <li class="post">
        <a href="/blog/2013/09/24/move-to-octopress/">Move to Octopress</a>
      </li>
    
      <li class="post">
        <a href="/blog/2013/09/03/nginx/">Nginx内核参数相关的优化设定说明</a>
      </li>
    
      <li class="post">
        <a href="/blog/2013/09/03/lamp/">LAMP系统性能调优</a>
      </li>
    
      <li class="post">
        <a href="/blog/2013/09/02/hello-jekyll-world/">Hello Jekyll World</a>
      </li>
    
      <li class="post">
        <a href="/blog/2013/02/23/%E4%BD%BF%E7%94%A8pry%E4%BB%A3%E6%9B%BFrails-console/">使用PRY代替RAILS CONSOLE</a>
      </li>
    
  </ul>
</section>
<section>
  
  <h1>Visitors</h1>
    <div id="clustrmaps-widget"></div><script type="text/javascript">
var _clustrmaps = {'url' : 'http://apiaceae.github.io', 'user' : 1114201, 'server' : '2', 'id' : 'clustrmaps-widget', 'version' : 1, 'date' : '2013-09-25', 'lang' : 'en', 'corners' : 'square' };
    (function ()
    { var s = document.createElement('script'); s.type = 'text/javascript'; s.async = true; s.src = 'http://www2.clustrmaps.com/counter/map.js'; var x = document.getElementsByTagName('script')[0]; x.parentNode.insertBefore(s, x);})();
    </script><noscript><a href="http://www2.clustrmaps.com/user/195110059"><img src="http://www2.clustrmaps.com/stats/maps-no_clusters/apiaceae.github.io-thumb.jpg" alt="Locations of visitors to this page"></a></noscript>

    <a href="http://info.flagcounter.com/wAs5"><img src="http://s07.flagcounter.com/count/wAs5/bg_FFFFFF/txt_000000/border_CCCCCC/columns_2/maxflags_12/viewers_0/labels_0/pageviews_0/flags_0/" alt="Flag Counter" border="0"></a>


</section>




  
</aside>

    </div>
  </div>
  <footer role="contentinfo"><p>
  Copyright &copy; 2013 - Hooker -
  <span class="credit">Powered by <a href="http://octopress.org">Octopress</a></span>
</p>

</footer>
  







  <script type="text/javascript">
    (function(){
      var twitterWidgets = document.createElement('script');
      twitterWidgets.type = 'text/javascript';
      twitterWidgets.async = true;
      twitterWidgets.src = '//platform.twitter.com/widgets.js';
      document.getElementsByTagName('head')[0].appendChild(twitterWidgets);
    })();
  </script>





</body>
</html>
