
<!DOCTYPE html>
<!--[if IEMobile 7 ]><html class="no-js iem7"><![endif]-->
<!--[if lt IE 9]><html class="no-js lte-ie8"><![endif]-->
<!--[if (gt IE 8)|(gt IEMobile 7)|!(IEMobile)|!(IE)]><!--><html class="no-js" lang="en"><!--<![endif]-->
<head>
  <meta charset="utf-8">
  <title>虎克的博客</title>
  <meta name="author" content="Hooker">

  
  <meta name="description" content="Book Description: Using theory, applications, and examples of inferences, Niche Modeling: Predictions from Statistical Distributions demonstrates &hellip;">
  

  <!-- http://t.co/dKP3o1e -->
  <meta name="HandheldFriendly" content="True">
  <meta name="MobileOptimized" content="320">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  
  <link rel="canonical" href="http://Apiaceae.github.io/blog/page/23/">
  <link href="/favicon.png" rel="icon">
  <link href="/stylesheets/screen.css" media="screen, projection" rel="stylesheet" type="text/css">
  <script src="/javascripts/modernizr-2.0.js"></script>
  <script src="/javascripts/ender.js"></script>
  <script src="/javascripts/octopress.js" type="text/javascript"></script>
  <link href="/atom.xml" rel="alternate" title="虎克的博客" type="application/atom+xml">
  <link href='http://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
  <!--Fonts from Google"s Web font directory at http://google.com/webfonts -->
<link href="http://fonts.googleapis.com/css?family=PT+Serif:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css">
<link href="http://fonts.googleapis.com/css?family=PT+Sans:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css">


  

</head>

<body   >
  <header role="banner"><hgroup>
  <div id="logo">
  	<div id="logoLeft">{</div>
  	<div id="logoText">mob</div>
  	<div id="logoRight">}</div>
  	<div class="clear"></div>
  </div>
  <h1><a href="/">虎克的博客</a></h1>
  
    <h2>Enthusiasm Biogeography-Biodiversity Informatics-Data Sciences</h2>
  
  <div class="clear"></div>
</hgroup>

</header>
  <nav role="navigation"><ul class="subscription" data-subscription="rss">
  <li><a href="/atom.xml" rel="subscribe-rss" title="subscribe via RSS">RSS</a></li>
  
</ul>
  
<form action="http://google.com/search" method="get">
  <fieldset role="search">
    <input type="hidden" name="q" value="site:Apiaceae.github.io" />
    <input class="search" type="text" name="q" results="0" placeholder="Search"/>
  </fieldset>
</form>
  
<ul class="main-navigation">
  <li><a href="/">Blog</a></li>
  <li><a href="/blog/archives">Archives</a></li>
  <li><a href="/tag-cloud">Tags</a></li>
  <li><a href="/about">About me</a></li>
  
</ul>

</nav>
  <div id="main">
    <div id="content">
      <div class="blog-index">
  
  
  
    <article>
      
  <header>
    
      <!-- Now we're back to normal posts. Note the links used under href in both headers.-->
        <h1 class="entry-title"><a href="/blog/2009/03/22/new-book-sdm/">生态位模型预测新书上架</a></h1>
      
    
    
      <p class="meta">
        








  


<time datetime="2009-03-22T00:00:00+08:00" pubdate data-updated="true">Mar 22<span>nd</span>, 2009</time>
        
        
                 | <a href="/blog/2009/03/22/new-book-sdm/#comments">Comments</a>
                
      </p>
    
  </header>


  <div class="entry-content"><p><img src="/images/2009/3/111.jpg" alt="书籍的封面" /></p>

<p><strong>Book Description</strong>: Using theory, applications, and examples of inferences, <strong>Niche Modeling</strong>: Predictions from Statistical Distributions demonstrates how to conduct and evaluate niche modeling projects in any area of application.</p>

<p>It features a series of theoretical and practical exercises for developing and evaluating niche models using the R statistics language. The author discusses applications of predictive modeling methods with reference to valid inferences from assumptions. He elucidates varied and simplified examples with rigor and completeness. Topics include geographic information systems, multivariate modeling, artificial intelligence methods, data handling, and information infrastructure.</p>

<p>Above all, successful niche modeling requires a deep understanding of the process of creating and using probability. Off-the-shelf statistical packages are tailored exactly to applications but can hide problematic complexities. Recipe book implementations fail to educate users in the details, assumptions, and pitfalls of analysis, but may be able to adapt to the specific needs of each study. Examining the sources of errors such as autocorrelation, bias, long term persistence, nonlinearity, circularity, and fraud, this seminal reference provides an understanding of the limitations and potential pitfalls of prediction, emphasizing the importance of avoiding errors.</p>

<p class='post-footer'>
       original link:
       <a href='http://Apiaceae.github.io/blog/2009/03/22/new-book-sdm/'>http://Apiaceae.github.io/blog/2009/03/22/new-book-sdm/</a><br/>
       &nbsp;written by <a href='http://Apiaceae.github.io'>Hooker</a>
       &nbsp;posted at <a href='http://Apiaceae.github.io'>http://Apiaceae.github.io</a>
       </p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <!-- Now we're back to normal posts. Note the links used under href in both headers.-->
        <h1 class="entry-title"><a href="/blog/2009/03/21/-lsid/">什么是生命科学识别码 LSID</a></h1>
      
    
    
      <p class="meta">
        








  


<time datetime="2009-03-21T00:00:00+08:00" pubdate data-updated="true">Mar 21<span>st</span>, 2009</time>
        
        
                 | <a href="/blog/2009/03/21/-lsid/#comments">Comments</a>
                
      </p>
    
  </header>


  <div class="entry-content"><p><img src="/images/2009/3/vtheme6.jpg" alt="LSID首页" /></p>

<p><strong>Life Science Identifiers (LSIDs)</strong> are persistent, location-independent, resource identifiers for uniquely naming biologically significant resources including species names, concepts, occurrences, genes or proteins, or data objects that encode information about them. To put it simply, LSIDs are a way to identify and locate pieces of biological information on the web.</p>

<p>The <em>LSID</em> concept introduces a straightforward approach to naming and identifying data resources stored in multiple, distributed data stores in a manner that overcomes the limitations of naming schemes in use today. Almost every public, internal, or department-level data store today has its own way of naming individual data resources, making integration between different data sources a tedious, never-ending chore for informatics developers and researchers.</p>

<p>By defining a simple, common way to identify and access biologically significant data, whether that data is stored in files, relational databases, in applications, or in internal or public data sources, LSID provides a naming standard underpinning for wide-area science and interoperability.</p>

<p>The Life Sciences Identifier is an <a href="http://www.omg.org/">Object Management Group (OMG)</a> Final Adopted Specification developed by the <a href="http://www.omg.org/lsr/">Life Sciences Research (LSR)</a>.</p>

<p class='post-footer'>
       original link:
       <a href='http://Apiaceae.github.io/blog/2009/03/21/-lsid/'>http://Apiaceae.github.io/blog/2009/03/21/-lsid/</a><br/>
       &nbsp;written by <a href='http://Apiaceae.github.io'>Hooker</a>
       &nbsp;posted at <a href='http://Apiaceae.github.io'>http://Apiaceae.github.io</a>
       </p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <!-- Now we're back to normal posts. Note the links used under href in both headers.-->
        <h1 class="entry-title"><a href="/blog/2009/03/20/american-nsfc-bi/">美国自然科学基金对数字化标本平台的支持</a></h1>
      
    
    
      <p class="meta">
        








  


<time datetime="2009-03-20T00:00:00+08:00" pubdate data-updated="true">Mar 20<span>th</span>, 2009</time>
        
        
                 | <a href="/blog/2009/03/20/american-nsfc-bi/#comments">Comments</a>
                
      </p>
    
  </header>


  <div class="entry-content"><p><img src="/images/2009/3/vtheme7.jpg" alt="项目首页" /></p>

<p>The US National Science Foundation has awarded a <strong>Research Coordination Grant</strong> entitled: <strong>Building a National Community of Natural History Collections</strong>.</p>

<p>The purpose of the RCN is to build a strong scientific community to better serve researchers in systematics, biodiversity, and other fields that use natural history collections.</p>

<p>Three professional organizations, <em>the American Institute of Biological Sciences</em>, <em>the Natural Science Collections Alliance</em>, and <em>the Society for the Preservation of Natural History Collections</em> will provide strong support as partners in the RCN, and many other societies have already built ties with the RCN by appointing Core Participants.</p>

<p>The RCN has the following objectives:</p>

<ul>
<li><p>Identify the institutions and people that will define our community and facilitate dialogue among them about how to better serve the needs of researchers,</p></li>
<li><p>identify major opportunities and challenges in the current environment and foreseeable future and develop a strategic plan for the future of collections research,</p></li>
<li><p>determine how to strengthen and modernize the role of collections in education and outreach, and</p></li>
<li><p>identify the primary needs of collections regarding care, curation, storage, and accessibility.</p></li>
</ul>


<p>The objectives will be met with a series of workshops, symposia, internships, and a website, <a href="http://www.collectionsweb.org/">www.CollectionsWeb.org</a>, which will provide a forum for interaction, host workshop reports, provide information on ways to become involved in the RCN, and provide links to resources for NHC.</p>

<p>Several major community resources will be developed, including a catalogue of NHC, a survey of the status of NHC, a register of curatorial expertise, and an inventory of innovative and successful educational programs.</p>

<p>Broadening minority participation in collections activities is a goal of the RCN.</p>

<p>The initial steering committee members are Henry L. Bart (Tulane University, vertebrate systematics), Meredith Blackwell (Louisiana State University, fungal systematics), L. Alan Prather (Michigan State University, plant
systematics) and James B. Woolley (Texas A &amp; M University, invertebrate systematics).</p>

<p>Announcements of upcoming events, opportunities to get involved, and reports will be posted to the website, so please check back often. Your input is essential!  Contact us at <a href="http://www.collectionsweb.org/about/steering_committee.html.">http://www.collectionsweb.org/about/steering_committee.html.</a></p>

<p class='post-footer'>
       original link:
       <a href='http://Apiaceae.github.io/blog/2009/03/20/american-nsfc-bi/'>http://Apiaceae.github.io/blog/2009/03/20/american-nsfc-bi/</a><br/>
       &nbsp;written by <a href='http://Apiaceae.github.io'>Hooker</a>
       &nbsp;posted at <a href='http://Apiaceae.github.io'>http://Apiaceae.github.io</a>
       </p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <!-- Now we're back to normal posts. Note the links used under href in both headers.-->
        <h1 class="entry-title"><a href="/blog/2009/03/19/%E5%9C%B0%E7%90%86%E4%BF%A1%E6%81%AF%E7%B3%BB%E7%BB%9F%E8%B5%84%E6%BA%90/">地理信息系统资源</a></h1>
      
    
    
      <p class="meta">
        








  


<time datetime="2009-03-19T03:35:23+08:00" pubdate data-updated="true">Mar 19<span>th</span>, 2009</time>
        
        
                 | <a href="/blog/2009/03/19/%E5%9C%B0%E7%90%86%E4%BF%A1%E6%81%AF%E7%B3%BB%E7%BB%9F%E8%B5%84%E6%BA%90/#comments">Comments</a>
                
      </p>
    
  </header>


  <div class="entry-content"><h2>All Types &amp; Geospatial</h2>


<p><ol></p>

<pre><code>&lt;li&gt;
&lt;div&gt;
&lt;h2 class="software_title"&gt;&lt;img class="icon" src="/bast_files/softwareTracker/Biomapper-moyen.gif" alt="" width="28" height="30" /&gt;&lt;a href="/node/67"&gt;Biomapper. A GIS-toolkit to model ecological niche and habitat suitability &lt;/a&gt;&lt;span class="reviewed_when"&gt;Wed, 10/17/2007 - 09:50&lt;/span&gt;&lt;/h2&gt;
&lt;p&gt;
Biomapperis a kit of GIS and statistical tools designed to build habitat suitability models and maps for any kind of animal or plant. It is centred on the Ecological Niche Factor Analysisthat allows to compute models without the need of absence data. 
&lt;/p&gt;
&lt;/div&gt;
&lt;/li&gt;
&lt;li&gt;
&lt;div&gt;
&lt;h2 class="software_title"&gt;&lt;img class="icon" src="/bast_files/softwareTracker/Music-icon.jpg" alt="" width="28" height="30" /&gt;&lt;a href="/node/64"&gt;CLIMEX &lt;/a&gt;&lt;span class="reviewed_when"&gt;Thu, 10/18/2007 - 13:01&lt;/span&gt;&lt;/h2&gt;
&lt;p&gt;
A commercial package that enables to asses the risk of pest establishing in a new location and the potential success or failure of a biological control agent from the current locations. 
&lt;/p&gt;
&lt;/div&gt;
&lt;/li&gt;
&lt;li&gt;
&lt;div&gt;
&lt;h2 class="software_title"&gt;&lt;img class="icon" src="/bast_files/softwareTracker/Music-icon.jpg" alt="" width="28" height="30" /&gt;&lt;a href="/node/56"&gt;deegree - Free Software for Spatial Data Infrastructures vdeegree 2.1-pre (recommended)&lt;/a&gt; &lt;span class="reviewed_when"&gt;Mon, 11/24/2008 - 13:23&lt;/span&gt;&lt;/h2&gt;
&lt;p&gt;
deegree is a Java Framework offering the main building blocks for Spatial Data Infrastructures. Its entire architecture is developed using standards of the Open Geospatial Consortium (OGC). 
&lt;/p&gt;
&lt;/div&gt;
&lt;/li&gt;
&lt;li&gt;
&lt;div&gt;
&lt;h2 class="software_title"&gt;&lt;img class="icon" src="/bast_files/softwareTracker/Music-icon.jpg" alt="" width="28" height="30" /&gt;&lt;a href="/node/49"&gt;DesktopGARP v1.1.6&lt;/a&gt; &lt;span class="reviewed_when"&gt;Wed, 10/17/2007 - 10:00&lt;/span&gt;&lt;/h2&gt;
&lt;p&gt;
DesktopGarp is a software package for biodiversity and ecologic research that allows the user to predict and analyze species distributions. GARP tries, interactively, to find non-random correlations between the presence and absence of the species and the values of the environmental parameters, using several types of rules. Each rule type implements a different method for building species prediction models. GARP as input takes enivonmental data and species occurrence localities. It generates random absence data. 
&lt;/p&gt;
&lt;/div&gt;
&lt;/li&gt;
&lt;li&gt;
&lt;div&gt;
&lt;h2 class="software_title"&gt;&lt;img class="icon" src="/bast_files/softwareTracker/logo.jpg" alt="" width="28" height="30" /&gt;&lt;a href="/node/23"&gt;DIVA-GIS v6.0&lt;/a&gt; &lt;span class="reviewed_when"&gt;Mon, 11/24/2008 - 13:29&lt;/span&gt;&lt;/h2&gt;
&lt;p&gt;
DIVA-GIS is a free and open source geographic information system (GIS) to make maps of species distribution data and analyze these data. DIVA-GIS was specifically developed at &lt;span class="nobr"&gt;&lt;a rel="nofollow" href="http://www.cipotato.org/"&gt;CIP&lt;sup&gt;&lt;img class="rendericon" src="http://research.cip.cgiar.org/confluence/images/icons/linkext7.gif" border="0" alt="" width="7" height="7" align="absMiddle" /&gt;&lt;/sup&gt;&lt;/a&gt;&lt;/span&gt; for use with genebank data such as available through national or international genebank documentation systems and &lt;span class="nobr"&gt;&lt;a rel="nofollow" href="http://singer.cgiar.org/"&gt;SINGER&lt;sup&gt;&lt;img class="rendericon" src="http://research.cip.cgiar.org/confluence/images/icons/linkext7.gif" border="0" alt="" width="7" height="7" align="absMiddle" /&gt;&lt;/sup&gt;&lt;/a&gt;&lt;/span&gt;. It consists many useful tools such as Grid Calculator (multiplying, adding rasters), Neighbourhood ( changing raster resolution) and Georeference Image. DIVA-GIS also has an Ecological Niche Modeling tool which can be used to predictive modeling ( it uses Bioclim and Domain algorithms). From DIVA-GIS desktop one can directly connect to DIVA-GIS Free GIS Data site and download climate grids, DEM, sattelite images or country level data. 
&lt;/p&gt;
&lt;/div&gt;
&lt;/li&gt;
&lt;li&gt;
&lt;div&gt;
&lt;h2 class="software_title"&gt;&lt;img class="icon" src="/bast_files/softwareTracker/Music-icon.jpg" alt="" width="28" height="30" /&gt;&lt;a href="/node/68"&gt;DIVA-GIS Annapurna &lt;/a&gt;&lt;span class="reviewed_when"&gt;Wed, 10/17/2007 - 10:01&lt;/span&gt;&lt;/h2&gt;
&lt;p&gt;
Another version of the same DIVA-GIS programme. 
&lt;/p&gt;
&lt;/div&gt;
&lt;/li&gt;
&lt;li&gt;
&lt;div&gt;
&lt;h2 class="software_title"&gt;&lt;img class="icon" src="/bast_files/softwareTracker/ecosim2.gif" alt="" width="28" height="30" /&gt;&lt;a href="/node/81"&gt;EcoSim. Null model software for ecologists &lt;/a&gt;&lt;span class="reviewed_when"&gt;Wed, 10/17/2007 - 10:04&lt;/span&gt;&lt;/h2&gt;
&lt;p&gt;
EcoSim allows you to test for community patterns with non-experimental data performing Monte Carlo randomizations to create &amp;ldquo;pseudo-communities&amp;rdquo;, then statistically compares the patterns in these randomized communities with those in the real data matrix. 
&lt;/p&gt;
&lt;/div&gt;
&lt;/li&gt;
&lt;li&gt;
&lt;div&gt;
&lt;h2 class="software_title"&gt;&lt;img class="icon" src="/bast_files/softwareTracker/EstimateSHeader.jpg" alt="" width="28" height="30" /&gt;&lt;a href="/node/84"&gt;EstimateS. Statistical estimation of species richness and shared species from samples &lt;/a&gt;&lt;span class="reviewed_when"&gt;Wed, 10/17/2007 - 10:07&lt;/span&gt;&lt;/h2&gt;
&lt;p&gt;
EstimateS is a free software application for Windows and Macintosh operating systems that computes a variety of biodiversity functions, estimators, and indices based on biotic sampling data. Some features require species relative abundance data, others only species presence/absence data. 
&lt;/p&gt;
&lt;/div&gt;
&lt;/li&gt;
&lt;li&gt;
&lt;div&gt;
&lt;h2 class="software_title"&gt;&lt;img class="icon" src="/bast_files/softwareTracker/Music-icon.jpg" alt="" width="28" height="30" /&gt;&lt;a href="/node/65"&gt;FloraMap &lt;/a&gt;&lt;span class="reviewed_when"&gt;Wed, 10/17/2007 - 10:09&lt;/span&gt;&lt;/h2&gt;
&lt;p&gt;
FloraMap is a software linked to agroclimatic and other databases able to showing the most likely distribution of wild species in nature. 
&lt;/p&gt;
&lt;/div&gt;
&lt;/li&gt;
&lt;li&gt;
&lt;div&gt;
&lt;h2 class="software_title"&gt;&lt;img class="icon" src="/bast_files/softwareTracker/gbif_05.jpg" alt="" width="28" height="30" /&gt;&lt;a href="/node/79"&gt;GBIF MAPA &lt;/a&gt;&lt;span class="reviewed_when"&gt;Wed, 10/17/2007 - 10:14&lt;/span&gt;&lt;/h2&gt;
&lt;p&gt;
A GBIF demonstration project which allows users to query the GBIF cache using names obtained through the Catalogue of life and to map and analyse the resultant record set. The Survey Gap Analysis (SGA) tool helps to design a biodiversity survey that will best complement the existing survey effort by identifying those areas least well surveyed in terms of environmental conditions. The Species Richness Assessment (SRA) tool provide an estimate of the number of species in an area; and to gain insight into the adequacy of sampling based on abundance distributions for those species. 
&lt;/p&gt;
&lt;/div&gt;
&lt;/li&gt;
&lt;li&gt;
&lt;div&gt;
&lt;h2 class="software_title"&gt;&lt;img class="icon" src="/bast_files/softwareTracker/Music-icon.jpg" alt="" width="28" height="30" /&gt;&lt;a href="/node/76"&gt;GeoDa. An Introduction to Spatial Data Analysis &lt;/a&gt;&lt;span class="reviewed_when"&gt;Thu, 10/18/2007 - 13:02&lt;/span&gt;&lt;/h2&gt;
&lt;p&gt;
GeoDa is the latest incarnation in a long line of software tools designed to implement techniques for exploratory spatial data analysis on lattice data (points and polygons). The free program provides a user friendly and graphical interface to methods of descriptive spatial data analysis, such as spatial autocorrelation statistics, as well as basic spatial regression functionality. 
&lt;/p&gt;
&lt;/div&gt;
&lt;/li&gt;
&lt;li&gt;
&lt;div&gt;
&lt;h2 class="software_title"&gt;&lt;img class="icon" src="/bast_files/softwareTracker/logomuseo.gif" alt="" width="28" height="30" /&gt;&lt;a href="/node/54"&gt;GIS data downloads &lt;/a&gt;&lt;span class="reviewed_when"&gt;Mon, 11/24/2008 - 13:28&lt;/span&gt;&lt;/h2&gt;
&lt;div&gt;
&lt;span style="font-size: small; font-family: Arial"&gt;&lt;font size="2"&gt;Download of Geographic Information Systems (GIS) data layers.&lt;/font&gt;&lt;/span&gt; 
&lt;/div&gt;
&lt;div&gt;
&lt;span style="font-size: small; font-family: Arial"&gt;&lt;font size="2"&gt;Free GIS layers from different sources, in Idrisi and Shape formats, etc&lt;p class='post-footer'&gt;
   original link:
   &lt;a href='http://Apiaceae.github.io/blog/2009/03/19/%E5%9C%B0%E7%90%86%E4%BF%A1%E6%81%AF%E7%B3%BB%E7%BB%9F%E8%B5%84%E6%BA%90/'&gt;http://Apiaceae.github.io/blog/2009/03/19/%E5%9C%B0%E7%90%86%E4%BF%A1%E6%81%AF%E7%B3%BB%E7%BB%9F%E8%B5%84%E6%BA%90/&lt;/a&gt;&lt;br/&gt;
   &amp;nbsp;written by &lt;a href='http://Apiaceae.github.io'&gt;Hooker&lt;/a&gt;
   &amp;nbsp;posted at &lt;a href='http://Apiaceae.github.io'&gt;http://Apiaceae.github.io&lt;/a&gt;
   &lt;/p&gt;
</code></pre>
</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <!-- Now we're back to normal posts. Note the links used under href in both headers.-->
        <h1 class="entry-title"><a href="/blog/2009/03/19/21-species-exploration/">21世纪的物种探险 Species Exploration</a></h1>
      
    
    
      <p class="meta">
        








  


<time datetime="2009-03-19T00:00:00+08:00" pubdate data-updated="true">Mar 19<span>th</span>, 2009</time>
        
        
                 | <a href="/blog/2009/03/19/21-species-exploration/#comments">Comments</a>
                
      </p>
    
  </header>


  <div class="entry-content"><p><img src="/images/2009/3/vtheme8.jpg" alt="Species Exploration首页" /></p>

<p><strong>Taxonomists</strong> discover, describe and classify <em>species</em>. Millions of species remain unknown or unidentifiable, inaccessible to science and society.</p>

<p>Charting the species of the world and their unique attributes are essential parts of understanding the <strong>history of life</strong>.</p>

<p>Reliable taxonomic information is a necessary for managing sustainable ecosystems, attaining conservation goals, and detecting introductions of pests, vectors and invasive species.</p>

<p><strong>Traditional taxonomic tools</strong> and methods have theoretical rigor and rich intellectual content, but are not keeping pace with this growing need for knowledge.</p>

<p><strong>How can we increase the pace at which species exploration progresses while maintaining the scientific rigor of traditional taxonomy?</strong></p>

<p class='post-footer'>
       original link:
       <a href='http://Apiaceae.github.io/blog/2009/03/19/21-species-exploration/'>http://Apiaceae.github.io/blog/2009/03/19/21-species-exploration/</a><br/>
       &nbsp;written by <a href='http://Apiaceae.github.io'>Hooker</a>
       &nbsp;posted at <a href='http://Apiaceae.github.io'>http://Apiaceae.github.io</a>
       </p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <!-- Now we're back to normal posts. Note the links used under href in both headers.-->
        <h1 class="entry-title"><a href="/blog/2009/03/18/%E4%BB%80%E4%B9%88%E6%98%AF%E6%A4%8D%E7%89%A9%E5%88%86%E7%B1%BB%E5%AD%A6-%E5%86%99%E5%9C%A8%E6%9E%97%E5%A5%88%E8%AF%9E%E8%BE%B0300%E5%B9%B4%E5%90%8E/">什么是植物分类学-写在林奈诞辰300年后</a></h1>
      
    
    
      <p class="meta">
        








  


<time datetime="2009-03-18T21:10:00+08:00" pubdate data-updated="true">Mar 18<span>th</span>, 2009</time>
        
        
                 | <a href="/blog/2009/03/18/%E4%BB%80%E4%B9%88%E6%98%AF%E6%A4%8D%E7%89%A9%E5%88%86%E7%B1%BB%E5%AD%A6-%E5%86%99%E5%9C%A8%E6%9E%97%E5%A5%88%E8%AF%9E%E8%BE%B0300%E5%B9%B4%E5%90%8E/#comments">Comments</a>
                
      </p>
    
  </header>


  <div class="entry-content"><p>
&nbsp;
</p>


<p>
<img src="/Blogs/image.axd?picture=2009%2f3%2fimage_db.jpg" alt="" width="369" height="450" />
</p>


<p>
植物分类是一门历史非常悠久的学科。它成为一门科学开始与16世纪的欧洲。林奈就是这么学科的鼻祖，他所创立的生物双命名法(属名+种加词=分类群名称),成为标示和交流生物有机体相关信息的基础好标准。尽管这门学科对其它应用性研究：比如生态学、生物多样性占据着举足轻重的作用，但这门学科在学术圈却遭到不同程度的贬低和歧视。在现代生物学科中又饱受争议的学科。 虽然对客观自然世界的事物进行分门别类是人作为高级生物与生俱来的一种能力。 自1735年《自然系统》(Systema Naturae)一书出版始,自然科学有了经典的分类学.该书作者林奈(C.von Linn&eacute;)的贡献在于,建立了人为分类体系和双名法则.《自然系统》的内容涉及植物、动物、矿物,历经270多年使用至今. 
</p>


<p>
植物分类有两个最为基本的理论环节： 
</p>


<h1>分类(Classification)：</h1>


<p>
分类即表明的是一种结果同时也是一个过程。 
</p>


<h1>命名(Nomenclature):</h1>


<p>
命名简而言之，对你所研究或关心的对象取个名字。 
</p>


<h1>鉴定 (Identification):</h1>


<p>
鉴定，简而言之，对已知的物种进行识别，判断它属于哪个物种，应该叫什么名字。 
</p>


<p class='post-footer'>
       original link:
       <a href='http://Apiaceae.github.io/blog/2009/03/18/%E4%BB%80%E4%B9%88%E6%98%AF%E6%A4%8D%E7%89%A9%E5%88%86%E7%B1%BB%E5%AD%A6-%E5%86%99%E5%9C%A8%E6%9E%97%E5%A5%88%E8%AF%9E%E8%BE%B0300%E5%B9%B4%E5%90%8E/'>http://Apiaceae.github.io/blog/2009/03/18/%E4%BB%80%E4%B9%88%E6%98%AF%E6%A4%8D%E7%89%A9%E5%88%86%E7%B1%BB%E5%AD%A6-%E5%86%99%E5%9C%A8%E6%9E%97%E5%A5%88%E8%AF%9E%E8%BE%B0300%E5%B9%B4%E5%90%8E/</a><br/>
       &nbsp;written by <a href='http://Apiaceae.github.io'>Hooker</a>
       &nbsp;posted at <a href='http://Apiaceae.github.io'>http://Apiaceae.github.io</a>
       </p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <!-- Now we're back to normal posts. Note the links used under href in both headers.-->
        <h1 class="entry-title"><a href="/blog/2009/03/18/Google+Map%E7%9A%84API%E5%BC%80%E5%8F%91%E5%AE%9E%E7%94%A8%E5%8A%9F%E8%83%BD/">Google Map的API开发实用功能</a></h1>
      
    
    
      <p class="meta">
        








  


<time datetime="2009-03-18T20:33:30+08:00" pubdate data-updated="true">Mar 18<span>th</span>, 2009</time>
        
        
                 | <a href="/blog/2009/03/18/Google+Map%E7%9A%84API%E5%BC%80%E5%8F%91%E5%AE%9E%E7%94%A8%E5%8A%9F%E8%83%BD/#comments">Comments</a>
                
      </p>
    
  </header>


  <div class="entry-content"><p>
<img src="/Blogs/image.axd?picture=2009%2f3%2f03.png" alt="" width="640" height="451" /> <img src="/Blogs/image.axd?picture=2009%2f3%2f002.png" alt="" width="640" height="466" />
</p>


<p>
详细的使用介绍见：<a href="http://code.google.com/p/donetgooglemapsapi/">http://code.google.com/p/donetgooglemapsapi/</a>
</p>


<p class='post-footer'>
       original link:
       <a href='http://Apiaceae.github.io/blog/2009/03/18/Google+Map%E7%9A%84API%E5%BC%80%E5%8F%91%E5%AE%9E%E7%94%A8%E5%8A%9F%E8%83%BD/'>http://Apiaceae.github.io/blog/2009/03/18/Google+Map%E7%9A%84API%E5%BC%80%E5%8F%91%E5%AE%9E%E7%94%A8%E5%8A%9F%E8%83%BD/</a><br/>
       &nbsp;written by <a href='http://Apiaceae.github.io'>Hooker</a>
       &nbsp;posted at <a href='http://Apiaceae.github.io'>http://Apiaceae.github.io</a>
       </p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <!-- Now we're back to normal posts. Note the links used under href in both headers.-->
        <h1 class="entry-title"><a href="/blog/2009/03/18/%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6%E7%AE%80%E6%8A%A5/">生物信息学简报</a></h1>
      
    
    
      <p class="meta">
        








  


<time datetime="2009-03-18T08:22:45+08:00" pubdate data-updated="true">Mar 18<span>th</span>, 2009</time>
        
        
                 | <a href="/blog/2009/03/18/%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6%E7%AE%80%E6%8A%A5/#comments">Comments</a>
                
      </p>
    
  </header>


  <div class="entry-content"><p>
<img src="/Blogs/image.axd?picture=2009%2f3%2f1.png" alt="" width="640" height="348" />
</p>


<p>
<em>Briefings in Bioinformatics</em> is an international forum for researchers and educators in the life sciences. The journal publishes reviews for the users of databases and analytical tools of contemporary genetics and molecular biology and is unique in providing practical help and guidance to the non-specialist.
</p>


<p>
<em>&quot;</em>Briefings in Bioinformatics <em>is a superb journal, and I find it to be an essential resource for staying current with the field of bioinformatics. The field of bioinformatics evolves at a rapid pace, and this journal rigorously describes current progress and often anticipates likely future developments. From a practical point of view, I use information from these articles in the laboratory and I also reference the journal in my teaching and publications.&quot;</em><br />
<br />
Jonathan Pevsner, Ph.D.<br />
Associate Professor, Department of Neurology<br />
Kennedy Krieger Institute, USA
</p>


<p>
期刊网址: <a href="http://bib.oxfordjournals.org/">http://bib.oxfordjournals.org/</a>
</p>


<p class='post-footer'>
       original link:
       <a href='http://Apiaceae.github.io/blog/2009/03/18/%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6%E7%AE%80%E6%8A%A5/'>http://Apiaceae.github.io/blog/2009/03/18/%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6%E7%AE%80%E6%8A%A5/</a><br/>
       &nbsp;written by <a href='http://Apiaceae.github.io'>Hooker</a>
       &nbsp;posted at <a href='http://Apiaceae.github.io'>http://Apiaceae.github.io</a>
       </p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <!-- Now we're back to normal posts. Note the links used under href in both headers.-->
        <h1 class="entry-title"><a href="/blog/2009/03/18/%E7%89%9B%E6%B4%A5%E5%A4%A7%E5%AD%A6%E7%9A%84%E6%96%B0%E6%9C%9F%E5%88%8A-%E6%95%B0%E6%8D%AE%E5%BA%93%28Database%29/">牛津大学的新期刊-数据库(Database)</a></h1>
      
    
    
      <p class="meta">
        








  


<time datetime="2009-03-18T08:06:39+08:00" pubdate data-updated="true">Mar 18<span>th</span>, 2009</time>
        
        
                 | <a href="/blog/2009/03/18/%E7%89%9B%E6%B4%A5%E5%A4%A7%E5%AD%A6%E7%9A%84%E6%96%B0%E6%9C%9F%E5%88%8A-%E6%95%B0%E6%8D%AE%E5%BA%93%28Database%29/#comments">Comments</a>
                
      </p>
    
  </header>


  <div class="entry-content"><p>
<img src="/Blogs/image.axd?picture=2009%2f3%2f001.png" alt="" width="640" height="348" /> 
</p>


<p>
Huge volumes of primary data are currently archived in numerous open-access databases, and with new generation technologies becoming more common in laboratories, large datasets will become even more prevalent than today. The archiving, curation, analysis and understanding of all of this data is a challenge. 
</p>


<p>
<em>
<h1>Database</h1></em>
</p>


<p>: <em>The Journal of Biological Databases and Curation</em> provides a platform for the presentation of novel ideas in database research surrounding biological information, and aims to help strengthen the bridge between database developers and users.</p>

<p>
期刊网址:http://www.oxfordjournals.org/our_journals/databa/&nbsp; 
</p>


<p>
&nbsp;
</p>


<p class='post-footer'>
       original link:
       <a href='http://Apiaceae.github.io/blog/2009/03/18/%E7%89%9B%E6%B4%A5%E5%A4%A7%E5%AD%A6%E7%9A%84%E6%96%B0%E6%9C%9F%E5%88%8A-%E6%95%B0%E6%8D%AE%E5%BA%93%28Database%29/'>http://Apiaceae.github.io/blog/2009/03/18/%E7%89%9B%E6%B4%A5%E5%A4%A7%E5%AD%A6%E7%9A%84%E6%96%B0%E6%9C%9F%E5%88%8A-%E6%95%B0%E6%8D%AE%E5%BA%93%28Database%29/</a><br/>
       &nbsp;written by <a href='http://Apiaceae.github.io'>Hooker</a>
       &nbsp;posted at <a href='http://Apiaceae.github.io'>http://Apiaceae.github.io</a>
       </p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <!-- Now we're back to normal posts. Note the links used under href in both headers.-->
        <h1 class="entry-title"><a href="/blog/2009/03/18/%E7%94%9F%E7%89%A9%E5%A4%9A%E6%A0%B7%E6%80%A7%E4%BF%A1%E6%81%AF%E5%AD%A6%E4%BB%8B%E7%BB%8D/">生物多样性信息学介绍</a></h1>
      
    
    
      <p class="meta">
        








  


<time datetime="2009-03-18T08:06:02+08:00" pubdate data-updated="true">Mar 18<span>th</span>, 2009</time>
        
        
                 | <a href="/blog/2009/03/18/%E7%94%9F%E7%89%A9%E5%A4%9A%E6%A0%B7%E6%80%A7%E4%BF%A1%E6%81%AF%E5%AD%A6%E4%BB%8B%E7%BB%8D/#comments">Comments</a>
                
      </p>
    
  </header>


  <div class="entry-content"><h1><img src="/Blogs/image.axd?picture=2009%2f3%2f2009-03-16_141550.png" alt="" width="430" height="41" /></h1>


<h1>An Overview of Biodiversity Informatics</h1>


<table border="0" cellspacing="2">
    <tbody>
        <tr>
            <td class="cleartable">Background for the <a href="http://www.all-species.org/meetings.html">first meeting</a> of the <strong><a href="http://www.all-species.org/">All Species Project</a></strong></td>
        </tr>
    </tbody>
</table>


<p>
<table border="0" cellspacing="0">
    <tbody>
        <tr>
            <td class="cleartable" align="right" valign="top">By</td>
            <td class="cleartable" valign="top">:&nbsp;</td>
            <td class="cleartable" valign="top">Stanley D. Blum </td>
        </tr>
        <tr>
            <td class="cleartable" align="right" valign="top">&nbsp;</td>
            <td class="cleartable" valign="top">&nbsp;</td>
            <td class="cleartable">California Academy of Sciences</td>
        </tr>
    </tbody>
</table>
</p>


<p>
<table border="0" cellspacing="0">
    <tbody>
        <tr>
            <td class="cleartable">First distributed</td>
            <td class="cleartable">&nbsp;:&nbsp;</td>
            <td class="cleartable">September 18, 2000</td>
        </tr>
        <tr>
            <td class="cleartable">This version</td>
            <td class="cleartable">&nbsp;:&nbsp;</td>
            <td class="cleartable">October 7, 2000</td>
        </tr>
    </tbody>
</table>
</p>


<h2>Preface</h2>


<p>
This overview of biodiversity informatics is strongly focussed on biological systematics and the work conducted in systematics collections (i.e., natural history museums). It does not address observational (monitoring) data, rare and endangered species, ecosystem characterization, threat assessment, and a host of other biodiversity issues. 
</p>


<h2>Introduction</h2>


<p>
Regardless of what the All Species project sets as its ultimate goals, it is certain that the work will create large quantities of new information. Most people will actually experience the project&#39;s results not by coming into contact with newly discovered and described organisms, but by experiencing the new information generated about the organisms &#8211; everything from text, to pictures, diagrams, maps, sounds and video. 
</p>


<p>
Ensuring that information flows efficiently, from creation, through analysis, into appropriate outputs, is the essence of biodiversity informatics &#8211; the application of information technology to the domain of biodiversity. 
</p>


<p>
This overview of biodiversity informatics describes the main subject areas in biological systematics, their interrelationships, and the most important informatics projects in a given area. The subject areas are: 
</p>


<ul>
    <li><a href="#names">Taxonomic Names and Classification</a> </li>
    <li><a href="#characters">Taxonomic Character Data: Taxonomic Descriptions, Keys, and Phylogenetic Data</a> </li>
    <li><a href="#specimens">Specimen Data and Species Distributions</a> </li>
</ul>


<p>
In each of these areas, I will discuss 
</p>


<ul>
    <li>The nature and uses of the information </li>
    <li>How the data are captured and managed </li>
    <li>Status of data capture and management within the community (e.g., percent digitized) </li>
    <li>Significant projects for compiling data, developing software, etc. </li>
</ul>


<h2><a name="names" title="names"></a>Taxonomic Names and Classification</h2>


<h4>Nature and Uses of Names and Classification</h4>


<p>
Biological taxonomy &#8211; the scientific names of organisms &#8211; provides a global (at least internationally recognized) system of designating natural groups of organisms; i.e., species and higher taxa. Classifications assemble smaller groups into larger groups, and provide a way of making statements about or retrieving information about many species at a time. One of the first steps in communicating the discovery of a new kind of organism is to give it a name, and to infer what it is by classifying it &#8211; i.e., saying that it is a particular kind of something more general and perhaps already familiar. 
</p>


<blockquote>
    <table border="0" cellspacing="0" cellpadding="6" style="width: 1520px; height: 85px">
        <tbody>
            <tr>
                <td class="txtbox" colspan="3"><strong>Examples of taxonomic names</strong></td>
            </tr>
            <tr>
                <td class="txtbox"><em>Gorilla gorilla</em></td>
                <td class="txtbox">the name of a species</td>
                <td class="txtbox">&nbsp;</td>
            </tr>
            <tr>
                <td class="txtbox" valign="top">Scombridae</td>
                <td class="txtbox">the name of a family of fishes (tunas, mackerels, etc.); the family contains 15 genera and 49 valid/accepted species; 212 species names are &quot;available&quot; for these 49 species so (212 - 49 =) 163 of those names are synonyms. </td>
                <td class="txtbox">&nbsp;</td>
            </tr>
            <tr>
                <td class="txtbox" align="right">&nbsp; </td>
                <td class="txtbox">
                <div align="right">
                <font size="-2">&#8211; (From the ITIS database)</font> 
                </div>
                </td>
                <td class="txtbox">&nbsp;</td>
            </tr>
            <tr>
                <td class="txtbox" colspan="3">&nbsp;</td>
            </tr>
            <tr>
                <td class="txtbox" colspan="3"><strong>An example of a species in a classification</strong></td>
            </tr>
            <tr>
                <td class="txtbox">&nbsp; </td>
                <td class="txtbox">
                <pre>
                                                                                Kingdom <strong>Animalia</strong> animals  Phylum <strong>Chordata</strong> chordates    Subphylum <strong>Vertebrata</strong> vertebrates      Class <strong>Mammalia</strong> mammals        Order <strong>Primates</strong> primates          Family <strong>Hominidae</strong> man-like primates            Genus <strong><em>Gorilla</em></strong>              Species <strong><em>Gorilla gorilla</em></strong>                                                              
                </pre>
                </td>
                <td class="txtbox">&nbsp;</td>
            </tr>
            <tr>
                <td class="txtbox" align="right">&nbsp; </td>
                <td class="txtbox" align="right"><font size="-2">&#8211; (From the ITIS database)</font></td>
                <td class="txtbox" align="right">&nbsp;</td>
            </tr>
        </tbody>
    </table>
</blockquote>


<p>
Defining a group of organisms (taxon) and naming it are conceptually distinct operations. Determining that a taxon is biologically meaningful or real is a question of scientific judgement and is subject to refutation. Determining what name should be applied to it is a matter of following the rules set forth in the relevant international code of nomenclature. (There are three; a code for microbiology, another for botany, and a third for zoology.) 
</p>


<p>
Taxonomic names are created and put into use via publication. There are no requirements of scientific veracity, reasonableness, or qualifications of the author for a name to be effectively published and admitted into the universe of discourse. Once a name is published it has to be dealt with in subsequent works. 
</p>


<h4>Scope of nomenclatural and classification data</h4>


<p>
There are an estimated 1.5 - 2 million known species. There are somewhere between one and two synonyms for every valid/accepted species (in addition the valid name). Compiling a list of scientific names for a major group takes years of effort. For the resulting list to represent real progress &#8211; something that doesn&#39;t have to be done again &#8211; each original publication must be reviewed (at least briefly) by a taxonomist and the decisions documented with supplemental information. Data gathered along with the name typically include the bibliographic reference, author(s), and date of publication. Additional information may include references to type specimens (institution and catalog number), type locality, and references to subsequent taxonomically significant publications. 
</p>


<h4>Significant projects</h4>


<p><p>
Projects that aim to compile taxonomic databases fall into two major categories: nomeclators and checklists. A nomenclator is a compilation of all relevant names, but does <em>not</em> present opinions about which taxa are accepted or valid. A checklist represents determinations about which taxa are accepted or v<p class='post-footer'></p>

<pre><code>   original link:
   &lt;a href='http://Apiaceae.github.io/blog/2009/03/18/%E7%94%9F%E7%89%A9%E5%A4%9A%E6%A0%B7%E6%80%A7%E4%BF%A1%E6%81%AF%E5%AD%A6%E4%BB%8B%E7%BB%8D/'&gt;http://Apiaceae.github.io/blog/2009/03/18/%E7%94%9F%E7%89%A9%E5%A4%9A%E6%A0%B7%E6%80%A7%E4%BF%A1%E6%81%AF%E5%AD%A6%E4%BB%8B%E7%BB%8D/&lt;/a&gt;&lt;br/&gt;
   &amp;nbsp;written by &lt;a href='http://Apiaceae.github.io'&gt;Hooker&lt;/a&gt;
   &amp;nbsp;posted at &lt;a href='http://Apiaceae.github.io'&gt;http://Apiaceae.github.io&lt;/a&gt;
   &lt;/p&gt;
</code></pre>
</div>
  
  


    </article>
  
  <div class="pagination">
    
      <a class="prev" href="/blog/page/24/">&larr; Older</a>
    
    <a href="/blog/archives">Blog Archives</a>
    
    <a class="next" href="/blog/page/22/">Newer &rarr;</a>
    
  </div>
</div>
<aside class="sidebar">
  
    <section>
  <img src="/images/profile.jpg" alt=""><br />
  <p>City: Beijing</p>
</section>
<section>
  <h2>What did I read</h2>
  <div>
    <script type="text/javascript" src="http://www.douban.com/service/badge/57046744/?show=collection&amp;select=random&amp;n=8&amp;columns=2&amp;cat=book" ></script>
    </div>
</section>
<section>
  <h1>Recent Posts</h1>
  <ul id="recent_posts">
    
      <li class="post">
        <a href="/blog/2013/09/26/cutom-table-css-from-sql-database/">读取数据库数据到HTMLTable 并加入一些漂亮的CSS</a>
      </li>
    
      <li class="post">
        <a href="/blog/2013/09/26/fifty-best-sites-build-on-ruby-on-rails/">最好的50个用Ruby on Rails开发的网站</a>
      </li>
    
      <li class="post">
        <a href="/blog/2013/09/26/why-we-live-in-big-city/">我？你？他？身处大城市的十大尴尬</a>
      </li>
    
      <li class="post">
        <a href="/blog/2013/09/23/pry-relace-rails-console/">使用PRY代替RAILS CONSOLE</a>
      </li>
    
      <li class="post">
        <a href="/blog/2013/09/03/nginx/">Nginx内核参数相关的优化设定说明</a>
      </li>
    
  </ul>
</section>
<section>
<h1>Recent Comments</h1>
<ul class="ds-recent-comments" data-num-items="5" data-show-avatars="1" data-show-time="1" data-show-title="1" data-show-admin="0" data-excerpt-length="18"></ul>

<!--多说js加载开始，一个页面只需要加载一次 -->
<script type="text/javascript">
  var duoshuoQuery = {short_name:"apiaceaegit"};
  (function() {
    var ds = document.createElement('script');
    ds.type = 'text/javascript';ds.async = true;
    ds.src = 'http://static.duoshuo.com/embed.js';
    ds.charset = 'UTF-8';
    (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(ds);
  })();
</script>
<!--多说js加载结束，一个页面只需要加载一次 -->

</section>
<section>
  <h1>Categories</h1>
  <ul id="categories">
    <li class='category'><a href='/blog/categories/asp-net/'>ASP.NET (4)</a></li>
<li class='category'><a href='/blog/categories/biodiversity/'>Biodiversity (1)</a></li>
<li class='category'><a href='/blog/categories/biodiversity-informatics/'>Biodiversity Informatics (5)</a></li>
<li class='category'><a href='/blog/categories/git/'>Git (1)</a></li>
<li class='category'><a href='/blog/categories/informatics/'>Informatics (1)</a></li>
<li class='category'><a href='/blog/categories/linux/'>Linux (8)</a></li>
<li class='category'><a href='/blog/categories/ruby/'>Ruby (1)</a></li>
<li class='category'><a href='/blog/categories/xml/'>XML (1)</a></li>
<li class='category'><a href='/blog/categories/data/'>data (1)</a></li>
<li class='category'><a href='/blog/categories/linux/'>linux (191)</a></li>
<li class='category'><a href='/blog/categories/ruby/'>ruby (1)</a></li>
<li class='category'><a href='/blog/categories/sciences/'>sciences (1)</a></li>
<li class='category'><a href='/blog/categories/web/'>web (1)</a></li>
<li class='category'><a href='/blog/categories/伞形科/'>伞形科 (1)</a></li>
<li class='category'><a href='/blog/categories/分类学/'>分类学 (5)</a></li>
<li class='category'><a href='/blog/categories/工具/'>工具 (1)</a></li>
<li class='category'><a href='/blog/categories/技巧/'>技巧 (1)</a></li>
<li class='category'><a href='/blog/categories/数据库/'>数据库 (1)</a></li>
<li class='category'><a href='/blog/categories/物种分布模型/'>物种分布模型 (1)</a></li>
<li class='category'><a href='/blog/categories/生物地理/'>生物地理 (2)</a></li>
<li class='category'><a href='/blog/categories/生物地理学/'>生物地理学 (3)</a></li>
<li class='category'><a href='/blog/categories/生物多样性信息学/'>生物多样性信息学 (7)</a></li>
<li class='category'><a href='/blog/categories/科学社会网络/'>科学社会网络 (1)</a></li>
<li class='category'><a href='/blog/categories/随笔/'>随笔 (4)</a></li>

  </ul>
</section>
<section>
  <h1>Tags</h1>
  <ul class="tag-cloud">
    <a style="font-size: 125%" href="/tags/asp-dot-net/">ASP.NET</a>
<a style="font-size: 180%" href="/tags/biodiversity-informatics/">Biodiversity Informatics</a>
<a style="font-size: 70%" href="/tags/lsid/">LSID</a>
<a style="font-size: 125%" href="/tags/linux/">Linux</a>
<a style="font-size: 70%" href="/tags/mapping/">Mapping</a>
<a style="font-size: 70%" href="/tags/sdm/">SDM</a>
<a style="font-size: 70%" href="/tags/concepts/">concepts</a>
<a style="font-size: 70%" href="/tags/etaxonomy/">etaxonomy</a>
<a style="font-size: 125%" href="/tags/sed/">sed</a>
<a style="font-size: 125%" href="/tags/taxonomy/">taxonomy</a>
<a style="font-size: 180%" href="/tags/ubuntu/">ubuntu</a>
<a style="font-size: 157%" href="/tags/xin-qing/">心情</a>
<a style="font-size: 157%" href="/tags/sheng-huo/">生活</a>
<a style="font-size: 180%" href="/tags/sheng-wu-duo-yang-xing-xin-xi-xue/">生物多样性信息学</a>
<a style="font-size: 157%" href="/tags/sui-bi/">随笔</a>

  </ul>
</section><section>
  
  <h1>Visitors</h1>
    <div id="clustrmaps-widget"></div><script type="text/javascript">
var _clustrmaps = {'url' : 'http://apiaceae.github.io', 'user' : 1114201, 'server' : '2', 'id' : 'clustrmaps-widget', 'version' : 1, 'date' : '2013-09-25', 'lang' : 'en', 'corners' : 'square' };
    (function ()
    { var s = document.createElement('script'); s.type = 'text/javascript'; s.async = true; s.src = 'http://www2.clustrmaps.com/counter/map.js'; var x = document.getElementsByTagName('script')[0]; x.parentNode.insertBefore(s, x);})();
    </script><noscript><a href="http://www2.clustrmaps.com/user/195110059"><img src="http://www2.clustrmaps.com/stats/maps-no_clusters/apiaceae.github.io-thumb.jpg" alt="Locations of visitors to this page"></a></noscript>

    <a href="http://info.flagcounter.com/wAs5"><img src="http://s07.flagcounter.com/count/wAs5/bg_FFFFFF/txt_000000/border_CCCCCC/columns_2/maxflags_12/viewers_0/labels_0/pageviews_0/flags_0/" alt="Flag Counter" border="0"></a>


</section>




  
</aside>

    </div>
  </div>
  <footer role="contentinfo"><p>
  Copyright &copy; 2013 - Hooker -
  <span class="credit">Powered by <a href="http://octopress.org">Octopress</a></span>
</p>

</footer>
  











</body>
</html>
